<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">

<head>
  <meta charset="utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1"/>
  <meta name="generator" content="distill" />

  <style type="text/css">
  /* Hide doc at startup (prevent jankiness while JS renders/transforms) */
  body {
    visibility: hidden;
  }
  </style>

 <!--radix_placeholder_import_source-->
 <!--/radix_placeholder_import_source-->

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ad0000; } /* Alert */
code span.an { color: #5e5e5e; } /* Annotation */
code span.at { } /* Attribute */
code span.bn { color: #ad0000; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007ba5; } /* ControlFlow */
code span.ch { color: #20794d; } /* Char */
code span.cn { color: #8f5902; } /* Constant */
code span.co { color: #5e5e5e; } /* Comment */
code span.cv { color: #5e5e5e; font-style: italic; } /* CommentVar */
code span.do { color: #5e5e5e; font-style: italic; } /* Documentation */
code span.dt { color: #ad0000; } /* DataType */
code span.dv { color: #ad0000; } /* DecVal */
code span.er { color: #ad0000; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #ad0000; } /* Float */
code span.fu { color: #4758ab; } /* Function */
code span.im { } /* Import */
code span.in { color: #5e5e5e; } /* Information */
code span.kw { color: #007ba5; } /* Keyword */
code span.op { color: #5e5e5e; } /* Operator */
code span.ot { color: #007ba5; } /* Other */
code span.pp { color: #ad0000; } /* Preprocessor */
code span.sc { color: #5e5e5e; } /* SpecialChar */
code span.ss { color: #20794d; } /* SpecialString */
code span.st { color: #20794d; } /* String */
code span.va { color: #111111; } /* Variable */
code span.vs { color: #20794d; } /* VerbatimString */
code span.wa { color: #5e5e5e; font-style: italic; } /* Warning */
</style>

<style>
  div.csl-bib-body { }
  div.csl-entry {
    clear: both;
    }
  .hanging div.csl-entry {
    margin-left:2em;
    text-indent:-2em;
  }
  div.csl-left-margin {
    min-width:2em;
    float:left;
  }
  div.csl-right-inline {
    margin-left:2em;
    padding-left:1em;
  }
  div.csl-indent {
    margin-left: 2em;
  }
</style>

  <!--radix_placeholder_meta_tags-->
  <title>Effects of mycorrhizal association on litter decomposition in the forest ecosystems: A meta-analysis</title>

  <meta property="description" itemprop="description" content="A short description of the post."/>


  <!--  https://schema.org/Article -->
  <meta property="article:published" itemprop="datePublished" content="2021-10-22"/>
  <meta property="article:created" itemprop="dateCreated" content="2021-10-22"/>
  <meta name="article:author" content="Meenu Patil"/>
  <meta name="article:author" content="Abhishek Kumar"/>
  <meta name="article:author" content="Pardeep Kumar"/>
  <meta name="article:author" content="Anand Narain Singh"/>

  <!--  https://developers.facebook.com/docs/sharing/webmasters#markup -->
  <meta property="og:title" content="Effects of mycorrhizal association on litter decomposition in the forest ecosystems: A meta-analysis"/>
  <meta property="og:type" content="article"/>
  <meta property="og:description" content="A short description of the post."/>
  <meta property="og:locale" content="en_US"/>

  <!--  https://dev.twitter.com/cards/types/summary -->
  <meta property="twitter:card" content="summary"/>
  <meta property="twitter:title" content="Effects of mycorrhizal association on litter decomposition in the forest ecosystems: A meta-analysis"/>
  <meta property="twitter:description" content="A short description of the post."/>

  <!--/radix_placeholder_meta_tags-->
  
  <meta name="citation_reference" content="citation_title=ape 5.0: An environment for modern phylogenetics and evolutionary analyses in R;citation_publication_date=2019;citation_volume=35;citation_author=E. Paradis;citation_author=K. Schliep"/>
  <meta name="citation_reference" content="citation_title=Introduction to meta-analysis;citation_publication_date=2009;citation_publisher=John Wiley &amp; Sons, Ltd;citation_doi=10.1002/9780470743386;citation_author=Michael Borenstein;citation_author=Larry V. Hedges;citation_author=Julian P. T. Higgins;citation_author=Hannah R. Rothstein"/>
  <meta name="citation_reference" content="citation_title=A basic introduction to fixed-effect and random-effects models for meta-analysis;citation_publication_date=2010;citation_publisher=Res Synth Methods;citation_volume=1;citation_doi=10.1002/jrsm.12;citation_issn=17592879;citation_author=Michael Borenstein;citation_author=Larry V. Hedges;citation_author=Julian P. T. Higgins;citation_author=Hannah R. Rothstein"/>
  <meta name="citation_reference" content="citation_title=Mycorrhizal type determines the magnitude and direction of root‐induced changes in decomposition in a temperate forest;citation_publication_date=2015;citation_publisher=Blackwell Publishing Ltd;citation_volume=206;citation_doi=10.1111/nph.13303;citation_issn=0028-646X;citation_author=Edward R. Brzostek;citation_author=Danilo Dragoni;citation_author=Zachary A. Brown;citation_author=Richard P. Phillips"/>
  <meta name="citation_reference" content="citation_title=Phylogenetic multilevel meta-analysis: A simulation study on the importance of modelling the phylogeny;citation_publication_date=2021;citation_publisher=Wiley;citation_volume=13;citation_doi=10.1111/2041-210x.13760;citation_author=Ozan Cinar;citation_author=Shinichi Nakagawa;citation_author=Wolfgang Viechtbauer"/>
  <meta name="citation_reference" content="citation_title=The combination of estimates from different experiments;citation_publication_date=1954;citation_publisher=JSTOR;citation_volume=10;citation_doi=10.2307/3001666;citation_issn=0006341X;citation_author=William G. Cochran"/>
  <meta name="citation_reference" content="citation_title=Plant species traits are the predominant control on litter decomposition rates within biomes worldwide;citation_publication_date=2008;citation_publisher=Wiley/Blackwell (10.1111);citation_volume=11;citation_doi=10.1111/j.1461-0248.2008.01219.x;citation_issn=1461023X;citation_author=William K. Cornwell;citation_author=Johannes H. C. C. Cornelissen;citation_author=Kathryn Amatangelo;citation_author=Ellen Dorrepaal;citation_author=Valerie T. Eviner;citation_author=Oscar Godoy;citation_author=Sarah E. Hobbie;citation_author=Bart Hoorens;citation_author=Hiroko Kurokawa;citation_author=Natalia Pérez-Harguindeguy;citation_author=Helen M. Quested;citation_author=Louis S. Santiago;citation_author=David A. Wardle;citation_author=Ian J. Wright;citation_author=Rien Aerts;citation_author=Steven D. Allison;citation_author=Peter Van Bodegom;citation_author=Victor Brovkin;citation_author=Alex Chatain;citation_author=Terry V. Callaghan;citation_author=Sandra Díaz;citation_author=Eric Garnier;citation_author=Diego E. Gurvich;citation_author=Elena Kazakou;citation_author=Julia A. Klein;citation_author=Jenny Read;citation_author=Peter B. Reich;citation_author=Nadejda A. Soudzilovskaia;citation_author=M. Victoria Vaieretti;citation_author=Mark Westoby"/>
  <meta name="citation_reference" content="citation_title=Litter decomposition and nutrient release in ectomycorrhizal mat soils of a Douglas fir ecosystem;citation_publication_date=1991;citation_volume=23;citation_doi=10.1016/0038-0717(91)90065-R;citation_issn=00380717;citation_author=James A Entry;citation_author=Cathy L Rose;citation_author=Kermtt Cromack"/>
  <meta name="citation_reference" content="citation_title=Revisiting the ‘Gadgil effect’: do interguild fungal interactions control carbon cycling in forest soils?;citation_publication_date=2016;citation_volume=209;citation_doi=10.1111/nph.13648;citation_issn=0028-646X;citation_author=Christopher W. Fernandez;citation_author=Peter G. Kennedy"/>
  <meta name="citation_reference" content="citation_title=Decelerated carbon cycling by ectomycorrhizal fungi is controlled by substrate quality and community composition;citation_publication_date=2020;citation_publisher=Blackwell Publishing Ltd;citation_volume=226;citation_doi=10.1111/nph.16269;citation_issn=0028-646X;citation_author=Christopher W. Fernandez;citation_author=Craig R. See;citation_author=Peter G. Kennedy"/>
  <meta name="citation_reference" content="citation_title=Mycorrhizal fungi as mediators of soil organic matter dynamics;citation_publication_date=2019;citation_publisher=Annual Reviews;citation_volume=50;citation_doi=10.1146/annurev-ecolsys-110617-062331;citation_author=Serita D. Frey"/>
  <meta name="citation_reference" content="citation_title=ggtree: An R package for visualization and annotation of phylogenetic trees with their covariates and other associated data;citation_publication_date=2016;citation_publisher=Wiley;citation_volume=8;citation_doi=10.1111/2041-210x.12628;citation_author=Guangchuang Yu;citation_author=David K. Smith;citation_author=Huachen Zhu;citation_author=Yi Guan;citation_author=Tommy Tsan-Yuk Lam"/>
  <meta name="citation_reference" content="citation_title=The phylogenetic regression;citation_publication_date=1989;citation_publisher=The Royal Society;citation_volume=326;citation_doi=10.1098/rstb.1989.0106;citation_author=Alan Grafen"/>
  <meta name="citation_reference" content="citation_title=Arbuscular mycorrhiza enhance the rate of litter decomposition while inhibiting soil microbial community development;citation_publication_date=2017;citation_publisher=Nature Publishing Group;citation_volume=7;citation_doi=10.1038/srep42184;citation_issn=2045-2322;citation_author=Heng Gui;citation_author=Kevin Hyde;citation_author=Jianchu Xu;citation_author=Peter Mortimer"/>
  <meta name="citation_reference" content="citation_title=Doing Meta-Analysis with R;citation_publication_date=2021;citation_publisher=Chapman; Hall/CRC;citation_doi=10.1201/9781003107347;citation_author=Mathias Harrer;citation_author=Pim Cuijpers;citation_author=Toshi A. Furukawa;citation_author=David D. Ebert"/>
  <meta name="citation_reference" content="citation_title=Interactions between an arbuscular mycorrhizal fungus and a soil microbial community mediating litter decomposition;citation_publication_date=2012;citation_publisher=Oxford Academic;citation_volume=80;citation_doi=10.1111/j.1574-6941.2011.01292.x;citation_issn=01686496;citation_author=Donald J. Herman;citation_author=Mary K. Firestone;citation_author=Erin Nuccio;citation_author=Angela Hodge"/>
  <meta name="citation_reference" content="citation_title=Quantifying heterogeneity in a meta-analysis;citation_publication_date=2002;citation_publisher=John Wiley &amp; Sons, Ltd;citation_volume=21;citation_doi=10.1002/sim.1186;citation_issn=02776715;citation_author=Julian P. T. Higgins;citation_author=Simon G. Thompson"/>
  <meta name="citation_reference" content="citation_title=An arbuscular mycorrhizal fungus accelerates decomposition and acquires nitrogen directly from organic material;citation_publication_date=2001;citation_publisher=Nature Publishing Group;citation_volume=413;citation_doi=10.1038/35095041;citation_issn=00280836;citation_author=Angela Hodge;citation_author=Colin D. Campbell;citation_author=Alastair H. Fitter"/>
  <meta name="citation_reference" content="citation_title=Rhizosphere carbon supply accelerates soil organic matter decomposition in the presence of fresh organic substrates;citation_publication_date=2019;citation_publisher=Springer;citation_volume=440;citation_doi=10.1007/s11104-019-04072-3;citation_issn=0032-079X;citation_author=Oyindamola Jackson;citation_author=Richard S. Quilliam;citation_author=Andy Stott;citation_author=Helen Grant;citation_author=Jens-Arne Subke"/>
  <meta name="citation_reference" content="citation_title=V.PhyloMaker: An R package that can generate very large phylogenies for vascular plants;citation_publication_date=2019;citation_publisher=Wiley;citation_volume=42;citation_doi=10.1111/ecog.04434;citation_author=Yi Jin;citation_author=Hong Qian"/>
  <meta name="citation_reference" content="citation_title=Leaf litter decay rates differ between mycorrhizal groups in temperate, but not tropical, forests;citation_publication_date=2019;citation_publisher=Wiley;citation_volume=222;citation_doi=10.1111/nph.15524;citation_author=Adrienne B. Keller;citation_author=Richard P. Phillips"/>
  <meta name="citation_reference" content="citation_title=Uses and misuses of meta-analysis in plant ecology;citation_publication_date=2014;citation_publisher=Wiley;citation_volume=102;citation_doi=10.1111/1365-2745.12224;citation_author=Julia Koricheva;citation_author=Jessica Gurevitch"/>
  <meta name="citation_reference" content="citation_title=Phosphorus and litter decomposability traits in tropical forest ecosystems under changing environment: A synthesis;citation_publication_date=2021;citation_publisher=Nova Science Publishers, Inc.;citation_author=Abhishek Kumar;citation_author=Meenu Patil;citation_author=Pardeep Kumar;citation_author=Anand Narain Singh"/>
  <meta name="citation_reference" content="citation_title=Fine roots and mycorrhizal fungi accelerate leaf litter decomposition in a northern hardwood forest regardless of dominant tree mycorrhizal associations;citation_publication_date=2021;citation_publisher=John Wiley &amp; Sons, Ltd;citation_volume=230;citation_doi=10.1111/nph.17155;citation_issn=0028-646X;citation_author=Ashley K. Lang;citation_author=Fiona V. Jevon;citation_author=Corinne R. Vietorisz;citation_author=Matthew P. Ayres;citation_author=Jaclyn Hatala Matthes"/>
  <meta name="citation_reference" content="citation_title=Do ectomycorrhizas alter leaf-litter decomposition in monodominant tropical forests of Guyana?;citation_publication_date=2006;citation_publisher=John Wiley &amp; Sons, Ltd;citation_volume=169;citation_doi=10.1111/j.1469-8137.2005.01607.x;citation_issn=0028646X;citation_author=Jordan R. Mayor;citation_author=Terry W. Henkel"/>
  <meta name="citation_reference" content="citation_title=Conducting Meta-Analyses in R with the metafor Package;citation_publication_date=2010;citation_volume=36;citation_doi=10.18637/jss.v036.i03;citation_issn=1548-7660;citation_author=Wolfgang Viechtbauer"/>
  <meta name="citation_reference" content="citation_title=Methodological issues and advances in biological meta-analysis;citation_publication_date=2012;citation_publisher=Springer Science; Business Media LLC;citation_volume=26;citation_doi=10.1007/s10682-012-9555-5;citation_author=Shinichi Nakagawa;citation_author=Eduardo S. A. Santos"/>
  <meta name="citation_reference" content="citation_title=Energy storage and the balance of producers and decomposers in ecological systems;citation_publication_date=1963;citation_volume=44;citation_doi=10.2307/1932179;citation_issn=0012-9658;citation_author=Jerry S. Olson"/>
  <meta name="citation_reference" content="citation_title=Comparative litter decomposability traits of selected native and exotic woody species from an urban environment of north-western siwalik region, india;citation_publication_date=2020;citation_publisher=Springer Science; Business Media LLC;citation_volume=10;citation_doi=10.1038/s41598-020-64576-2;citation_author=Meenu Patil;citation_author=Abhishek Kumar;citation_author=Pardeep Kumar;citation_author=Navneet Kaur Cheema;citation_author=Rupinder Kaur;citation_author=Ramchand Bhatti;citation_author=A. N. Singh"/>
  <meta name="citation_reference" content="citation_title=An updated megaphylogeny of plants, a tool for generating plant phylogenies and an analysis of phylogenetic community structure;citation_publication_date=2016;citation_publisher=Oxford University Press (OUP);citation_volume=9;citation_doi=10.1093/jpe/rtv047;citation_author=Hong Qian;citation_author=Yi Jin"/>
  <meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;citation_publication_date=2021;citation_publisher=R Foundation for Statistical Computing;citation_author=R: A language and environment for statistical computing"/>
  <meta name="citation_reference" content="citation_title=Carbon availability triggers the decomposition of plant litter and assimilation of nitrogen by an ectomycorrhizal fungus;citation_publication_date=2013;citation_volume=7;citation_doi=10.1038/ismej.2013.91;citation_issn=1751-7362;citation_author=F Rineau;citation_author=F Shah;citation_author=M M Smits;citation_author=P Persson;citation_author=T Johansson;citation_author=R Carleer;citation_author=C Troein;citation_author=A Tunlid"/>
  <meta name="citation_reference" content="citation_title=Who is who in litter decomposition Metaproteomics reveals major microbial players and their biogeochemical functions;citation_publication_date=2012;citation_publisher=Nature Publishing Group;citation_volume=6;citation_doi=10.1038/ismej.2012.11;citation_issn=17517362;citation_author=Thomas Schneider;citation_author=Katharina M. Keiblinger;citation_author=Emanuel Schmid;citation_author=Katja Sterflinger-Gleixner;citation_author=Günther Ellersdorfer;citation_author=Bernd Roschitzki;citation_author=Andreas Richter;citation_author=Leo Eberl;citation_author=Sophie Zechmeister-Boltenstern;citation_author=Kathrin Riedel"/>
  <meta name="citation_reference" content="citation_title=Heterogeneity in ecological and evolutionary meta-analyses: Its magnitude and implications;citation_publication_date=2016;citation_publisher=Wiley;citation_volume=97;citation_doi=10.1002/ecy.1591;citation_author=Alistair M. Senior;citation_author=Catherine E. Grueber;citation_author=Tsukushi Kamiya;citation_author=Malgorzata Lagisz;citation_author=Katie ODwyer;citation_author=Eduardo S. A. Santos;citation_author=Shinichi Nakagawa"/>
  <meta name="citation_reference" content="citation_title=Ectomycorrhizal fungi decompose soil organic matter using oxidative mechanisms adapted from saprotrophic ancestors;citation_publication_date=2016;citation_publisher=John Wiley &amp; Sons, Ltd;citation_volume=209;citation_doi=10.1111/NPH.13722;citation_issn=1469-8137;citation_author=Firoz Shah;citation_author=César Nicolás;citation_author=Johan Bentzer;citation_author=Magnus Ellström;citation_author=Mark Smits;citation_author=Francois Rineau;citation_author=Björn Canbäck;citation_author=Dimitrios Floudas;citation_author=Robert Carleer;citation_author=Gerald Lackner;citation_author=Jana Braesel;citation_author=Dirk Hoffmeister;citation_author=Bernard Henrissat;citation_author=Dag Ahrén;citation_author=Tomas Johansson;citation_author=David S. Hibbett;citation_author=Francis Martin;citation_author=Per Persson;citation_author=Anders Tunlid"/>
  <meta name="citation_reference" content="citation_title=Constructing a broadly inclusive seed plant phylogeny;citation_publication_date=2018;citation_publisher=Wiley;citation_volume=105;citation_doi=10.1002/ajb2.1019;citation_author=Stephen A. Smith;citation_author=Joseph W. Brown"/>
  <meta name="citation_reference" content="citation_title=Contrasting effects of ectomycorrhizal fungi on early and late stage decomposition in a boreal forest;citation_publication_date=2018;citation_volume=12;citation_doi=10.1038/s41396-018-0181-2;citation_issn=1751-7362;citation_author=Erica Sterkenburg;citation_author=Karina E. Clemmensen;citation_author=Alf Ekblad;citation_author=Roger D. Finlay;citation_author=Björn D. Lindahl"/>
  <meta name="citation_reference" content="citation_title=Bias and efficiency of meta-analytic variance estimators in the random-effects model;citation_publication_date=2005;citation_publisher=American Educational Research Association (AERA);citation_volume=30;citation_doi=10.3102/10769986030003261;citation_author=Wolfgang Viechtbauer"/>
  <meta name="citation_reference" content="citation_title=synthesisr: Import, assemble, and deduplicate bibliographic datasets;citation_publication_date=2020;citation_publisher=CRAN;citation_author=Martin Westgate;citation_author=Eliza Grames"/>
  <meta name="citation_reference" content="citation_title=Welcome to the Tidyverse;citation_publication_date=2019;citation_volume=4;citation_doi=10.21105/joss.01686;citation_issn=2475-9066;citation_author=Hadley Wickham;citation_author=Mara Averick;citation_author=Jennifer Bryan;citation_author=Winston Chang;citation_author=Lucy McGowan;citation_author=Romain François;citation_author=Garrett Grolemund;citation_author=Alex Hayes;citation_author=Lionel Henry;citation_author=Jim Hester;citation_author=Max Kuhn;citation_author=Thomas Pedersen;citation_author=Evan Miller;citation_author=Stephan Bache;citation_author=Kirill Müller;citation_author=Jeroen Ooms;citation_author=David Robinson;citation_author=Dana Seidel;citation_author=Vitalie Spinu;citation_author=Kohske Takahashi;citation_author=Davis Vaughan;citation_author=Claus Wilke;citation_author=Kara Woo;citation_author=Hiroaki Yutani"/>
  <meta name="citation_reference" content="citation_title=Three keys to the radiation of angiosperms into freezing environments;citation_publication_date=2014;citation_publisher=Springer Science; Business Media LLC;citation_volume=506;citation_doi=10.1038/nature12872;citation_author=Amy E. Zanne;citation_author=David C. Tank;citation_author=William K. Cornwell;citation_author=Jonathan M. Eastman;citation_author=Stephen A. Smith;citation_author=Richard G. FitzJohn;citation_author=Daniel J. McGlinn;citation_author=Brian C. O’Meara;citation_author=Angela T. Moles;citation_author=Peter B. Reich;citation_author=Dana L. Royer;citation_author=Douglas E. Soltis;citation_author=Peter F. Stevens;citation_author=Mark Westoby;citation_author=Ian J. Wright;citation_author=Lonnie Aarssen;citation_author=Robert I. Bertin;citation_author=Andre Calaminus;citation_author=Rafaël Govaerts;citation_author=Frank Hemmings;citation_author=Michelle R. Leishman;citation_author=Jacek Oleksyn;citation_author=Pamela S. Soltis;citation_author=Nathan G. Swenson;citation_author=Laura Warman;citation_author=Jeremy M. Beaulieu"/>
  <meta name="citation_reference" content="citation_title=Rates of litter decomposition in terrestrial ecosystems: global patterns and controlling factors;citation_publication_date=2008;citation_publisher=Narnia;citation_volume=1;citation_doi=10.1093/jpe/rtn002;citation_issn=1752-9921;citation_author=Deqiang Zhang;citation_author=Dafeng Hui;citation_author=Yiqi Luo;citation_author=Guoyi Zhou"/>
  <meta name="citation_reference" content="citation_title=The effects of mycorrhizal roots on litter decomposition, soil biota, and nutrients in a spodosolic soil;citation_publication_date=1996;citation_publisher=Kluwer Academic Publishers;citation_volume=179;citation_doi=10.1007/BF00011648;citation_issn=0032-079X;citation_author=Weixing Zhu;citation_author=Joan G. Ehrenfeld"/>
  <!--radix_placeholder_rmarkdown_metadata-->

  <script type="text/json" id="radix-rmarkdown-metadata">
  {"type":"list","attributes":{"names":{"type":"character","attributes":{},"value":["name","title","description","author","date","bibliography","csl","draft","output"]}},"value":[{"type":"character","attributes":{},"value":["meta-analysis"]},{"type":"character","attributes":{},"value":["Effects of mycorrhizal association on litter decomposition in the forest ecosystems: A meta-analysis"]},{"type":"character","attributes":{},"value":["A short description of the post.\n"]},{"type":"list","attributes":{},"value":[{"type":"list","attributes":{"names":{"type":"character","attributes":{},"value":["name","url","affiliation","affiliation_url","orcid_id"]}},"value":[{"type":"character","attributes":{},"value":["Meenu Patil"]},{"type":"character","attributes":{},"value":["https://www.researchgate.net/profile/Meenu-Patil"]},{"type":"character","attributes":{},"value":["Panjab University, Chandigarh"]},{"type":"character","attributes":{},"value":["https://puchd.ac.in"]},{"type":"character","attributes":{},"value":["0000-0002-7664-7877"]}]},{"type":"list","attributes":{"names":{"type":"character","attributes":{},"value":["name","url","affiliation","affiliation_url","orcid_id"]}},"value":[{"type":"character","attributes":{},"value":["Abhishek Kumar"]},{"type":"character","attributes":{},"value":["https://akumar.netlify.app"]},{"type":"character","attributes":{},"value":["Panjab University, Chandigarh"]},{"type":"character","attributes":{},"value":["https://puchd.ac.in"]},{"type":"character","attributes":{},"value":["0000-0003-2252-7623"]}]},{"type":"list","attributes":{"names":{"type":"character","attributes":{},"value":["name","url","affiliation","affiliation_url"]}},"value":[{"type":"character","attributes":{},"value":["Pardeep Kumar"]},{"type":"character","attributes":{},"value":["https://www.researchgate.net/profile/Pardeep-Kumar-22"]},{"type":"character","attributes":{},"value":["Panjab University, Chandigarh"]},{"type":"character","attributes":{},"value":["https://puchd.ac.in"]}]},{"type":"list","attributes":{"names":{"type":"character","attributes":{},"value":["name","url","affiliation","affiliation_url","orcid_id"]}},"value":[{"type":"character","attributes":{},"value":["Anand Narain Singh"]},{"type":"character","attributes":{},"value":["https://www.researchgate.net/profile/Anand-Singh-15"]},{"type":"character","attributes":{},"value":["Panjab University, Chandigarh"]},{"type":"character","attributes":{},"value":["https://puchd.ac.in"]},{"type":"character","attributes":{},"value":["0000-0002-0148-8680"]}]}]},{"type":"character","attributes":{},"value":["2021-10-22"]},{"type":"character","attributes":{},"value":["refs.bib"]},{"type":"character","attributes":{},"value":["mycorrhiza.csl"]},{"type":"logical","attributes":{},"value":[true]},{"type":"list","attributes":{"names":{"type":"character","attributes":{},"value":["distill::distill_article"]}},"value":[{"type":"list","attributes":{"names":{"type":"character","attributes":{},"value":["self_contained","toc","toc_depth","toc_float","theme"]}},"value":[{"type":"logical","attributes":{},"value":[false]},{"type":"logical","attributes":{},"value":[true]},{"type":"integer","attributes":{},"value":[3]},{"type":"logical","attributes":{},"value":[true]},{"type":"character","attributes":{},"value":["pygments"]}]}]}]}
  </script>
  <!--/radix_placeholder_rmarkdown_metadata-->
  
  <script type="text/json" id="radix-resource-manifest">
  {"type":"character","attributes":{},"value":["apa.csl","data/bib/bib.zip","data/bib/gs-md.bib","data/bib/gs-mna.bib","data/bib/scopus.bib","data/bib/uniqueref.ris","data/bib/wos1.bib","data/bib/wos2.bib","data/bib/wos3.bib","data/Fig1.svg","data/mass_remaining.csv","data/phylo.tre","data/raw_entry1991.csv","data/species_classification.csv","data/study_metadata.csv","meta-analysis_files/anchor-4.2.2/anchor.min.js","meta-analysis_files/bowser-1.9.3/bowser.min.js","meta-analysis_files/DiagrammeR-styles-0.2/styles.css","meta-analysis_files/distill-2.2.21/template.v2.js","meta-analysis_files/figure-html5/fig2-1.svg","meta-analysis_files/figure-html5/fig3-1.svg","meta-analysis_files/figure-html5/fig4-1.svg","meta-analysis_files/figure-html5/fig5-1.svg","meta-analysis_files/figure-html5/fig6-1.svg","meta-analysis_files/figure-html5/out-inf-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-10-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-11-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-12-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-12-2.svg","meta-analysis_files/figure-html5/unnamed-chunk-12-3.svg","meta-analysis_files/figure-html5/unnamed-chunk-13-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-13-2.svg","meta-analysis_files/figure-html5/unnamed-chunk-14-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-14-2.svg","meta-analysis_files/figure-html5/unnamed-chunk-14-3.svg","meta-analysis_files/figure-html5/unnamed-chunk-15-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-15-2.svg","meta-analysis_files/figure-html5/unnamed-chunk-16-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-16-2.svg","meta-analysis_files/figure-html5/unnamed-chunk-2-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-4-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-5-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-5-2.svg","meta-analysis_files/figure-html5/unnamed-chunk-6-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-7-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-7-2.svg","meta-analysis_files/figure-html5/unnamed-chunk-8-1.svg","meta-analysis_files/figure-html5/unnamed-chunk-9-1.svg","meta-analysis_files/grViz-binding-1.0.6.1/grViz.js","meta-analysis_files/header-attrs-2.11/header-attrs.js","meta-analysis_files/htmlwidgets-1.5.4/htmlwidgets.js","meta-analysis_files/jquery-3.6.0/jquery-3.6.0.js","meta-analysis_files/jquery-3.6.0/jquery-3.6.0.min.js","meta-analysis_files/jquery-3.6.0/jquery-3.6.0.min.map","meta-analysis_files/popper-2.6.0/popper.min.js","meta-analysis_files/tippy-6.2.7/tippy-bundle.umd.min.js","meta-analysis_files/tippy-6.2.7/tippy-light-border.css","meta-analysis_files/tippy-6.2.7/tippy.css","meta-analysis_files/tippy-6.2.7/tippy.umd.min.js","meta-analysis_files/viz-1.8.2/viz.js","meta-analysis_files/webcomponents-2.0.0/webcomponents.js","mycorrhiza.csl","refs.bib"]}
  </script>
  <!--radix_placeholder_navigation_in_header-->
  <!--/radix_placeholder_navigation_in_header-->
  <!--radix_placeholder_distill-->

  <style type="text/css">

  body {
    background-color: white;
  }

  .pandoc-table {
    width: 100%;
  }

  .pandoc-table>caption {
    margin-bottom: 10px;
  }

  .pandoc-table th:not([align]) {
    text-align: left;
  }

  .pagedtable-footer {
    font-size: 15px;
  }

  d-byline .byline {
    grid-template-columns: 2fr 2fr;
  }

  d-byline .byline h3 {
    margin-block-start: 1.5em;
  }

  d-byline .byline .authors-affiliations h3 {
    margin-block-start: 0.5em;
  }

  .authors-affiliations .orcid-id {
    width: 16px;
    height:16px;
    margin-left: 4px;
    margin-right: 4px;
    vertical-align: middle;
    padding-bottom: 2px;
  }

  d-title .dt-tags {
    margin-top: 1em;
    grid-column: text;
  }

  .dt-tags .dt-tag {
    text-decoration: none;
    display: inline-block;
    color: rgba(0,0,0,0.6);
    padding: 0em 0.4em;
    margin-right: 0.5em;
    margin-bottom: 0.4em;
    font-size: 70%;
    border: 1px solid rgba(0,0,0,0.2);
    border-radius: 3px;
    text-transform: uppercase;
    font-weight: 500;
  }

  d-article table.gt_table td,
  d-article table.gt_table th {
    border-bottom: none;
  }

  .html-widget {
    margin-bottom: 2.0em;
  }

  .l-screen-inset {
    padding-right: 16px;
  }

  .l-screen .caption {
    margin-left: 10px;
  }

  .shaded {
    background: rgb(247, 247, 247);
    padding-top: 20px;
    padding-bottom: 20px;
    border-top: 1px solid rgba(0, 0, 0, 0.1);
    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
  }

  .shaded .html-widget {
    margin-bottom: 0;
    border: 1px solid rgba(0, 0, 0, 0.1);
  }

  .shaded .shaded-content {
    background: white;
  }

  .text-output {
    margin-top: 0;
    line-height: 1.5em;
  }

  .hidden {
    display: none !important;
  }

  d-article {
    padding-top: 2.5rem;
    padding-bottom: 30px;
  }

  d-appendix {
    padding-top: 30px;
  }

  d-article>p>img {
    width: 100%;
  }

  d-article h2 {
    margin: 1rem 0 1.5rem 0;
  }

  d-article h3 {
    margin-top: 1.5rem;
  }

  d-article iframe {
    border: 1px solid rgba(0, 0, 0, 0.1);
    margin-bottom: 2.0em;
    width: 100%;
  }

  /* Tweak code blocks */

  d-article div.sourceCode code,
  d-article pre code {
    font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
  }

  d-article pre,
  d-article div.sourceCode,
  d-article div.sourceCode pre {
    overflow: auto;
  }

  d-article div.sourceCode {
    background-color: white;
  }

  d-article div.sourceCode pre {
    padding-left: 10px;
    font-size: 12px;
    border-left: 2px solid rgba(0,0,0,0.1);
  }

  d-article pre {
    font-size: 12px;
    color: black;
    background: none;
    margin-top: 0;
    text-align: left;
    white-space: pre;
    word-spacing: normal;
    word-break: normal;
    word-wrap: normal;
    line-height: 1.5;

    -moz-tab-size: 4;
    -o-tab-size: 4;
    tab-size: 4;

    -webkit-hyphens: none;
    -moz-hyphens: none;
    -ms-hyphens: none;
    hyphens: none;
  }

  d-article pre a {
    border-bottom: none;
  }

  d-article pre a:hover {
    border-bottom: none;
    text-decoration: underline;
  }

  d-article details {
    grid-column: text;
    margin-bottom: 0.8em;
  }

  @media(min-width: 768px) {

  d-article pre,
  d-article div.sourceCode,
  d-article div.sourceCode pre {
    overflow: visible !important;
  }

  d-article div.sourceCode pre {
    padding-left: 18px;
    font-size: 14px;
  }

  d-article pre {
    font-size: 14px;
  }

  }

  figure img.external {
    background: white;
    border: 1px solid rgba(0, 0, 0, 0.1);
    box-shadow: 0 1px 8px rgba(0, 0, 0, 0.1);
    padding: 18px;
    box-sizing: border-box;
  }

  /* CSS for d-contents */

  .d-contents {
    grid-column: text;
    color: rgba(0,0,0,0.8);
    font-size: 0.9em;
    padding-bottom: 1em;
    margin-bottom: 1em;
    padding-bottom: 0.5em;
    margin-bottom: 1em;
    padding-left: 0.25em;
    justify-self: start;
  }

  @media(min-width: 1000px) {
    .d-contents.d-contents-float {
      height: 0;
      grid-column-start: 1;
      grid-column-end: 4;
      justify-self: center;
      padding-right: 3em;
      padding-left: 2em;
    }
  }

  .d-contents nav h3 {
    font-size: 18px;
    margin-top: 0;
    margin-bottom: 1em;
  }

  .d-contents li {
    list-style-type: none
  }

  .d-contents nav > ul {
    padding-left: 0;
  }

  .d-contents ul {
    padding-left: 1em
  }

  .d-contents nav ul li {
    margin-top: 0.6em;
    margin-bottom: 0.2em;
  }

  .d-contents nav a {
    font-size: 13px;
    border-bottom: none;
    text-decoration: none
    color: rgba(0, 0, 0, 0.8);
  }

  .d-contents nav a:hover {
    text-decoration: underline solid rgba(0, 0, 0, 0.6)
  }

  .d-contents nav > ul > li > a {
    font-weight: 600;
  }

  .d-contents nav > ul > li > ul {
    font-weight: inherit;
  }

  .d-contents nav > ul > li > ul > li {
    margin-top: 0.2em;
  }


  .d-contents nav ul {
    margin-top: 0;
    margin-bottom: 0.25em;
  }

  .d-article-with-toc h2:nth-child(2) {
    margin-top: 0;
  }


  /* Figure */

  .figure {
    position: relative;
    margin-bottom: 2.5em;
    margin-top: 1.5em;
  }

  .figure img {
    width: 100%;
  }

  .figure .caption {
    color: rgba(0, 0, 0, 0.6);
    font-size: 12px;
    line-height: 1.5em;
  }

  .figure img.external {
    background: white;
    border: 1px solid rgba(0, 0, 0, 0.1);
    box-shadow: 0 1px 8px rgba(0, 0, 0, 0.1);
    padding: 18px;
    box-sizing: border-box;
  }

  .figure .caption a {
    color: rgba(0, 0, 0, 0.6);
  }

  .figure .caption b,
  .figure .caption strong, {
    font-weight: 600;
    color: rgba(0, 0, 0, 1.0);
  }

  /* Citations */

  d-article .citation {
    color: inherit;
    cursor: inherit;
  }

  div.hanging-indent{
    margin-left: 1em; text-indent: -1em;
  }

  /* Citation hover box */

  .tippy-box[data-theme~=light-border] {
    background-color: rgba(250, 250, 250, 0.95);
  }

  .tippy-content > p {
    margin-bottom: 0;
    padding: 2px;
  }


  /* Tweak 1000px media break to show more text */

  @media(min-width: 1000px) {
    .base-grid,
    distill-header,
    d-title,
    d-abstract,
    d-article,
    d-appendix,
    distill-appendix,
    d-byline,
    d-footnote-list,
    d-citation-list,
    distill-footer {
      grid-template-columns: [screen-start] 1fr [page-start kicker-start] 80px [middle-start] 50px [text-start kicker-end] 65px 65px 65px 65px 65px 65px 65px 65px [text-end gutter-start] 65px [middle-end] 65px [page-end gutter-end] 1fr [screen-end];
      grid-column-gap: 16px;
    }

    .grid {
      grid-column-gap: 16px;
    }

    d-article {
      font-size: 1.06rem;
      line-height: 1.7em;
    }
    figure .caption, .figure .caption, figure figcaption {
      font-size: 13px;
    }
  }

  @media(min-width: 1180px) {
    .base-grid,
    distill-header,
    d-title,
    d-abstract,
    d-article,
    d-appendix,
    distill-appendix,
    d-byline,
    d-footnote-list,
    d-citation-list,
    distill-footer {
      grid-template-columns: [screen-start] 1fr [page-start kicker-start] 60px [middle-start] 60px [text-start kicker-end] 60px 60px 60px 60px 60px 60px 60px 60px [text-end gutter-start] 60px [middle-end] 60px [page-end gutter-end] 1fr [screen-end];
      grid-column-gap: 32px;
    }

    .grid {
      grid-column-gap: 32px;
    }
  }


  /* Get the citation styles for the appendix (not auto-injected on render since
     we do our own rendering of the citation appendix) */

  d-appendix .citation-appendix,
  .d-appendix .citation-appendix {
    font-size: 11px;
    line-height: 15px;
    border-left: 1px solid rgba(0, 0, 0, 0.1);
    padding-left: 18px;
    border: 1px solid rgba(0,0,0,0.1);
    background: rgba(0, 0, 0, 0.02);
    padding: 10px 18px;
    border-radius: 3px;
    color: rgba(150, 150, 150, 1);
    overflow: hidden;
    margin-top: -12px;
    white-space: pre-wrap;
    word-wrap: break-word;
  }

  /* Include appendix styles here so they can be overridden */

  d-appendix {
    contain: layout style;
    font-size: 0.8em;
    line-height: 1.7em;
    margin-top: 60px;
    margin-bottom: 0;
    border-top: 1px solid rgba(0, 0, 0, 0.1);
    color: rgba(0,0,0,0.5);
    padding-top: 60px;
    padding-bottom: 48px;
  }

  d-appendix h3 {
    grid-column: page-start / text-start;
    font-size: 15px;
    font-weight: 500;
    margin-top: 1em;
    margin-bottom: 0;
    color: rgba(0,0,0,0.65);
  }

  d-appendix h3 + * {
    margin-top: 1em;
  }

  d-appendix ol {
    padding: 0 0 0 15px;
  }

  @media (min-width: 768px) {
    d-appendix ol {
      padding: 0 0 0 30px;
      margin-left: -30px;
    }
  }

  d-appendix li {
    margin-bottom: 1em;
  }

  d-appendix a {
    color: rgba(0, 0, 0, 0.6);
  }

  d-appendix > * {
    grid-column: text;
  }

  d-appendix > d-footnote-list,
  d-appendix > d-citation-list,
  d-appendix > distill-appendix {
    grid-column: screen;
  }

  /* Include footnote styles here so they can be overridden */

  d-footnote-list {
    contain: layout style;
  }

  d-footnote-list > * {
    grid-column: text;
  }

  d-footnote-list a.footnote-backlink {
    color: rgba(0,0,0,0.3);
    padding-left: 0.5em;
  }



  /* Anchor.js */

  .anchorjs-link {
    /*transition: all .25s linear; */
    text-decoration: none;
    border-bottom: none;
  }
  *:hover > .anchorjs-link {
    margin-left: -1.125em !important;
    text-decoration: none;
    border-bottom: none;
  }

  /* Social footer */

  .social_footer {
    margin-top: 30px;
    margin-bottom: 0;
    color: rgba(0,0,0,0.67);
  }

  .disqus-comments {
    margin-right: 30px;
  }

  .disqus-comment-count {
    border-bottom: 1px solid rgba(0, 0, 0, 0.4);
    cursor: pointer;
  }

  #disqus_thread {
    margin-top: 30px;
  }

  .article-sharing a {
    border-bottom: none;
    margin-right: 8px;
  }

  .article-sharing a:hover {
    border-bottom: none;
  }

  .sidebar-section.subscribe {
    font-size: 12px;
    line-height: 1.6em;
  }

  .subscribe p {
    margin-bottom: 0.5em;
  }


  .article-footer .subscribe {
    font-size: 15px;
    margin-top: 45px;
  }


  .sidebar-section.custom {
    font-size: 12px;
    line-height: 1.6em;
  }

  .custom p {
    margin-bottom: 0.5em;
  }

  /* Styles for listing layout (hide title) */
  .layout-listing d-title, .layout-listing .d-title {
    display: none;
  }

  /* Styles for posts lists (not auto-injected) */


  .posts-with-sidebar {
    padding-left: 45px;
    padding-right: 45px;
  }

  .posts-list .description h2,
  .posts-list .description p {
    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif;
  }

  .posts-list .description h2 {
    font-weight: 700;
    border-bottom: none;
    padding-bottom: 0;
  }

  .posts-list h2.post-tag {
    border-bottom: 1px solid rgba(0, 0, 0, 0.2);
    padding-bottom: 12px;
  }
  .posts-list {
    margin-top: 60px;
    margin-bottom: 24px;
  }

  .posts-list .post-preview {
    text-decoration: none;
    overflow: hidden;
    display: block;
    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
    padding: 24px 0;
  }

  .post-preview-last {
    border-bottom: none !important;
  }

  .posts-list .posts-list-caption {
    grid-column: screen;
    font-weight: 400;
  }

  .posts-list .post-preview h2 {
    margin: 0 0 6px 0;
    line-height: 1.2em;
    font-style: normal;
    font-size: 24px;
  }

  .posts-list .post-preview p {
    margin: 0 0 12px 0;
    line-height: 1.4em;
    font-size: 16px;
  }

  .posts-list .post-preview .thumbnail {
    box-sizing: border-box;
    margin-bottom: 24px;
    position: relative;
    max-width: 500px;
  }
  .posts-list .post-preview img {
    width: 100%;
    display: block;
  }

  .posts-list .metadata {
    font-size: 12px;
    line-height: 1.4em;
    margin-bottom: 18px;
  }

  .posts-list .metadata > * {
    display: inline-block;
  }

  .posts-list .metadata .publishedDate {
    margin-right: 2em;
  }

  .posts-list .metadata .dt-authors {
    display: block;
    margin-top: 0.3em;
    margin-right: 2em;
  }

  .posts-list .dt-tags {
    display: block;
    line-height: 1em;
  }

  .posts-list .dt-tags .dt-tag {
    display: inline-block;
    color: rgba(0,0,0,0.6);
    padding: 0.3em 0.4em;
    margin-right: 0.2em;
    margin-bottom: 0.4em;
    font-size: 60%;
    border: 1px solid rgba(0,0,0,0.2);
    border-radius: 3px;
    text-transform: uppercase;
    font-weight: 500;
  }

  .posts-list img {
    opacity: 1;
  }

  .posts-list img[data-src] {
    opacity: 0;
  }

  .posts-more {
    clear: both;
  }


  .posts-sidebar {
    font-size: 16px;
  }

  .posts-sidebar h3 {
    font-size: 16px;
    margin-top: 0;
    margin-bottom: 0.5em;
    font-weight: 400;
    text-transform: uppercase;
  }

  .sidebar-section {
    margin-bottom: 30px;
  }

  .categories ul {
    list-style-type: none;
    margin: 0;
    padding: 0;
  }

  .categories li {
    color: rgba(0, 0, 0, 0.8);
    margin-bottom: 0;
  }

  .categories li>a {
    border-bottom: none;
  }

  .categories li>a:hover {
    border-bottom: 1px solid rgba(0, 0, 0, 0.4);
  }

  .categories .active {
    font-weight: 600;
  }

  .categories .category-count {
    color: rgba(0, 0, 0, 0.4);
  }


  @media(min-width: 768px) {
    .posts-list .post-preview h2 {
      font-size: 26px;
    }
    .posts-list .post-preview .thumbnail {
      float: right;
      width: 30%;
      margin-bottom: 0;
    }
    .posts-list .post-preview .description {
      float: left;
      width: 45%;
    }
    .posts-list .post-preview .metadata {
      float: left;
      width: 20%;
      margin-top: 8px;
    }
    .posts-list .post-preview p {
      margin: 0 0 12px 0;
      line-height: 1.5em;
      font-size: 16px;
    }
    .posts-with-sidebar .posts-list {
      float: left;
      width: 75%;
    }
    .posts-with-sidebar .posts-sidebar {
      float: right;
      width: 20%;
      margin-top: 60px;
      padding-top: 24px;
      padding-bottom: 24px;
    }
  }


  /* Improve display for browsers without grid (IE/Edge <= 15) */

  .downlevel {
    line-height: 1.6em;
    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif;
    margin: 0;
  }

  .downlevel .d-title {
    padding-top: 6rem;
    padding-bottom: 1.5rem;
  }

  .downlevel .d-title h1 {
    font-size: 50px;
    font-weight: 700;
    line-height: 1.1em;
    margin: 0 0 0.5rem;
  }

  .downlevel .d-title p {
    font-weight: 300;
    font-size: 1.2rem;
    line-height: 1.55em;
    margin-top: 0;
  }

  .downlevel .d-byline {
    padding-top: 0.8em;
    padding-bottom: 0.8em;
    font-size: 0.8rem;
    line-height: 1.8em;
  }

  .downlevel .section-separator {
    border: none;
    border-top: 1px solid rgba(0, 0, 0, 0.1);
  }

  .downlevel .d-article {
    font-size: 1.06rem;
    line-height: 1.7em;
    padding-top: 1rem;
    padding-bottom: 2rem;
  }


  .downlevel .d-appendix {
    padding-left: 0;
    padding-right: 0;
    max-width: none;
    font-size: 0.8em;
    line-height: 1.7em;
    margin-bottom: 0;
    color: rgba(0,0,0,0.5);
    padding-top: 40px;
    padding-bottom: 48px;
  }

  .downlevel .footnotes ol {
    padding-left: 13px;
  }

  .downlevel .base-grid,
  .downlevel .distill-header,
  .downlevel .d-title,
  .downlevel .d-abstract,
  .downlevel .d-article,
  .downlevel .d-appendix,
  .downlevel .distill-appendix,
  .downlevel .d-byline,
  .downlevel .d-footnote-list,
  .downlevel .d-citation-list,
  .downlevel .distill-footer,
  .downlevel .appendix-bottom,
  .downlevel .posts-container {
    padding-left: 40px;
    padding-right: 40px;
  }

  @media(min-width: 768px) {
    .downlevel .base-grid,
    .downlevel .distill-header,
    .downlevel .d-title,
    .downlevel .d-abstract,
    .downlevel .d-article,
    .downlevel .d-appendix,
    .downlevel .distill-appendix,
    .downlevel .d-byline,
    .downlevel .d-footnote-list,
    .downlevel .d-citation-list,
    .downlevel .distill-footer,
    .downlevel .appendix-bottom,
    .downlevel .posts-container {
    padding-left: 150px;
    padding-right: 150px;
    max-width: 900px;
  }
  }

  .downlevel pre code {
    display: block;
    border-left: 2px solid rgba(0, 0, 0, .1);
    padding: 0 0 0 20px;
    font-size: 14px;
  }

  .downlevel code, .downlevel pre {
    color: black;
    background: none;
    font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
    text-align: left;
    white-space: pre;
    word-spacing: normal;
    word-break: normal;
    word-wrap: normal;
    line-height: 1.5;

    -moz-tab-size: 4;
    -o-tab-size: 4;
    tab-size: 4;

    -webkit-hyphens: none;
    -moz-hyphens: none;
    -ms-hyphens: none;
    hyphens: none;
  }

  .downlevel .posts-list .post-preview {
    color: inherit;
  }



  </style>

  <script type="application/javascript">

  function is_downlevel_browser() {
    if (bowser.isUnsupportedBrowser({ msie: "12", msedge: "16"},
                                   window.navigator.userAgent)) {
      return true;
    } else {
      return window.load_distill_framework === undefined;
    }
  }

  // show body when load is complete
  function on_load_complete() {

    // add anchors
    if (window.anchors) {
      window.anchors.options.placement = 'left';
      window.anchors.add('d-article > h2, d-article > h3, d-article > h4, d-article > h5');
    }


    // set body to visible
    document.body.style.visibility = 'visible';

    // force redraw for leaflet widgets
    if (window.HTMLWidgets) {
      var maps = window.HTMLWidgets.findAll(".leaflet");
      $.each(maps, function(i, el) {
        var map = this.getMap();
        map.invalidateSize();
        map.eachLayer(function(layer) {
          if (layer instanceof L.TileLayer)
            layer.redraw();
        });
      });
    }

    // trigger 'shown' so htmlwidgets resize
    $('d-article').trigger('shown');
  }

  function init_distill() {

    init_common();

    // create front matter
    var front_matter = $('<d-front-matter></d-front-matter>');
    $('#distill-front-matter').wrap(front_matter);

    // create d-title
    $('.d-title').changeElementType('d-title');

    // create d-byline
    var byline = $('<d-byline></d-byline>');
    $('.d-byline').replaceWith(byline);

    // create d-article
    var article = $('<d-article></d-article>');
    $('.d-article').wrap(article).children().unwrap();

    // move posts container into article
    $('.posts-container').appendTo($('d-article'));

    // create d-appendix
    $('.d-appendix').changeElementType('d-appendix');

    // flag indicating that we have appendix items
    var appendix = $('.appendix-bottom').children('h3').length > 0;

    // replace footnotes with <d-footnote>
    $('.footnote-ref').each(function(i, val) {
      appendix = true;
      var href = $(this).attr('href');
      var id = href.replace('#', '');
      var fn = $('#' + id);
      var fn_p = $('#' + id + '>p');
      fn_p.find('.footnote-back').remove();
      var text = fn_p.html();
      var dtfn = $('<d-footnote></d-footnote>');
      dtfn.html(text);
      $(this).replaceWith(dtfn);
    });
    // remove footnotes
    $('.footnotes').remove();

    // move refs into #references-listing
    $('#references-listing').replaceWith($('#refs'));

    $('h1.appendix, h2.appendix').each(function(i, val) {
      $(this).changeElementType('h3');
    });
    $('h3.appendix').each(function(i, val) {
      var id = $(this).attr('id');
      $('.d-contents a[href="#' + id + '"]').parent().remove();
      appendix = true;
      $(this).nextUntil($('h1, h2, h3')).addBack().appendTo($('d-appendix'));
    });

    // show d-appendix if we have appendix content
    $("d-appendix").css('display', appendix ? 'grid' : 'none');

    // localize layout chunks to just output
    $('.layout-chunk').each(function(i, val) {

      // capture layout
      var layout = $(this).attr('data-layout');

      // apply layout to markdown level block elements
      var elements = $(this).children().not('details, div.sourceCode, pre, script');
      elements.each(function(i, el) {
        var layout_div = $('<div class="' + layout + '"></div>');
        if (layout_div.hasClass('shaded')) {
          var shaded_content = $('<div class="shaded-content"></div>');
          $(this).wrap(shaded_content);
          $(this).parent().wrap(layout_div);
        } else {
          $(this).wrap(layout_div);
        }
      });


      // unwrap the layout-chunk div
      $(this).children().unwrap();
    });

    // remove code block used to force  highlighting css
    $('.distill-force-highlighting-css').parent().remove();

    // remove empty line numbers inserted by pandoc when using a
    // custom syntax highlighting theme
    $('code.sourceCode a:empty').remove();

    // load distill framework
    load_distill_framework();

    // wait for window.distillRunlevel == 4 to do post processing
    function distill_post_process() {

      if (!window.distillRunlevel || window.distillRunlevel < 4)
        return;

      // hide author/affiliations entirely if we have no authors
      var front_matter = JSON.parse($("#distill-front-matter").html());
      var have_authors = front_matter.authors && front_matter.authors.length > 0;
      if (!have_authors)
        $('d-byline').addClass('hidden');

      // article with toc class
      $('.d-contents').parent().addClass('d-article-with-toc');

      // strip links that point to #
      $('.authors-affiliations').find('a[href="#"]').removeAttr('href');

      // add orcid ids
      $('.authors-affiliations').find('.author').each(function(i, el) {
        var orcid_id = front_matter.authors[i].orcidID;
        if (orcid_id) {
          var a = $('<a></a>');
          a.attr('href', 'https://orcid.org/' + orcid_id);
          var img = $('<img></img>');
          img.addClass('orcid-id');
          img.attr('alt', 'ORCID ID');
          img.attr('src','data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg==');
          a.append(img);
          $(this).append(a);
        }
      });

      // hide elements of author/affiliations grid that have no value
      function hide_byline_column(caption) {
        $('d-byline').find('h3:contains("' + caption + '")').parent().css('visibility', 'hidden');
      }

      // affiliations
      var have_affiliations = false;
      for (var i = 0; i<front_matter.authors.length; ++i) {
        var author = front_matter.authors[i];
        if (author.affiliation !== "&nbsp;") {
          have_affiliations = true;
          break;
        }
      }
      if (!have_affiliations)
        $('d-byline').find('h3:contains("Affiliations")').css('visibility', 'hidden');

      // published date
      if (!front_matter.publishedDate)
        hide_byline_column("Published");

      // document object identifier
      var doi = $('d-byline').find('h3:contains("DOI")');
      var doi_p = doi.next().empty();
      if (!front_matter.doi) {
        // if we have a citation and valid citationText then link to that
        if ($('#citation').length > 0 && front_matter.citationText) {
          doi.html('Citation');
          $('<a href="#citation"></a>')
            .text(front_matter.citationText)
            .appendTo(doi_p);
        } else {
          hide_byline_column("DOI");
        }
      } else {
        $('<a></a>')
           .attr('href', "https://doi.org/" + front_matter.doi)
           .html(front_matter.doi)
           .appendTo(doi_p);
      }

       // change plural form of authors/affiliations
      if (front_matter.authors.length === 1) {
        var grid = $('.authors-affiliations');
        grid.children('h3:contains("Authors")').text('Author');
        grid.children('h3:contains("Affiliations")').text('Affiliation');
      }

      // remove d-appendix and d-footnote-list local styles
      $('d-appendix > style:first-child').remove();
      $('d-footnote-list > style:first-child').remove();

      // move appendix-bottom entries to the bottom
      $('.appendix-bottom').appendTo('d-appendix').children().unwrap();
      $('.appendix-bottom').remove();

      // hoverable references
      $('span.citation[data-cites]').each(function() {
        var refs = $(this).attr('data-cites').split(" ");
        var refHtml = refs.map(function(ref) {
          return "<p>" + $('#ref-' + ref).html() + "</p>";
        }).join("\n");
        window.tippy(this, {
          allowHTML: true,
          content: refHtml,
          maxWidth: 500,
          interactive: true,
          interactiveBorder: 10,
          theme: 'light-border',
          placement: 'bottom-start'
        });
      });

      // clear polling timer
      clearInterval(tid);

      // show body now that everything is ready
      on_load_complete();
    }

    var tid = setInterval(distill_post_process, 50);
    distill_post_process();

  }

  function init_downlevel() {

    init_common();

     // insert hr after d-title
    $('.d-title').after($('<hr class="section-separator"/>'));

    // check if we have authors
    var front_matter = JSON.parse($("#distill-front-matter").html());
    var have_authors = front_matter.authors && front_matter.authors.length > 0;

    // manage byline/border
    if (!have_authors)
      $('.d-byline').remove();
    $('.d-byline').after($('<hr class="section-separator"/>'));
    $('.d-byline a').remove();

    // remove toc
    $('.d-contents').remove();

    // move appendix elements
    $('h1.appendix, h2.appendix').each(function(i, val) {
      $(this).changeElementType('h3');
    });
    $('h3.appendix').each(function(i, val) {
      $(this).nextUntil($('h1, h2, h3')).addBack().appendTo($('.d-appendix'));
    });


    // inject headers into references and footnotes
    var refs_header = $('<h3></h3>');
    refs_header.text('References');
    $('#refs').prepend(refs_header);

    var footnotes_header = $('<h3></h3');
    footnotes_header.text('Footnotes');
    $('.footnotes').children('hr').first().replaceWith(footnotes_header);

    // move appendix-bottom entries to the bottom
    $('.appendix-bottom').appendTo('.d-appendix').children().unwrap();
    $('.appendix-bottom').remove();

    // remove appendix if it's empty
    if ($('.d-appendix').children().length === 0)
      $('.d-appendix').remove();

    // prepend separator above appendix
    $('.d-appendix').before($('<hr class="section-separator" style="clear: both"/>'));

    // trim code
    $('pre>code').each(function(i, val) {
      $(this).html($.trim($(this).html()));
    });

    // move posts-container right before article
    $('.posts-container').insertBefore($('.d-article'));

    $('body').addClass('downlevel');

    on_load_complete();
  }


  function init_common() {

    // jquery plugin to change element types
    (function($) {
      $.fn.changeElementType = function(newType) {
        var attrs = {};

        $.each(this[0].attributes, function(idx, attr) {
          attrs[attr.nodeName] = attr.nodeValue;
        });

        this.replaceWith(function() {
          return $("<" + newType + "/>", attrs).append($(this).contents());
        });
      };
    })(jQuery);

    // prevent underline for linked images
    $('a > img').parent().css({'border-bottom' : 'none'});

    // mark non-body figures created by knitr chunks as 100% width
    $('.layout-chunk').each(function(i, val) {
      var figures = $(this).find('img, .html-widget');
      if ($(this).attr('data-layout') !== "l-body") {
        figures.css('width', '100%');
      } else {
        figures.css('max-width', '100%');
        figures.filter("[width]").each(function(i, val) {
          var fig = $(this);
          fig.css('width', fig.attr('width') + 'px');
        });

      }
    });

    // auto-append index.html to post-preview links in file: protocol
    // and in rstudio ide preview
    $('.post-preview').each(function(i, val) {
      if (window.location.protocol === "file:")
        $(this).attr('href', $(this).attr('href') + "index.html");
    });

    // get rid of index.html references in header
    if (window.location.protocol !== "file:") {
      $('.distill-site-header a[href]').each(function(i,val) {
        $(this).attr('href', $(this).attr('href').replace("index.html", "./"));
      });
    }

    // add class to pandoc style tables
    $('tr.header').parent('thead').parent('table').addClass('pandoc-table');
    $('.kable-table').children('table').addClass('pandoc-table');

    // add figcaption style to table captions
    $('caption').parent('table').addClass("figcaption");

    // initialize posts list
    if (window.init_posts_list)
      window.init_posts_list();

    // implmement disqus comment link
    $('.disqus-comment-count').click(function() {
      window.headroom_prevent_pin = true;
      $('#disqus_thread').toggleClass('hidden');
      if (!$('#disqus_thread').hasClass('hidden')) {
        var offset = $(this).offset();
        $(window).resize();
        $('html, body').animate({
          scrollTop: offset.top - 35
        });
      }
    });
  }

  document.addEventListener('DOMContentLoaded', function() {
    if (is_downlevel_browser())
      init_downlevel();
    else
      window.addEventListener('WebComponentsReady', init_distill);
  });

  </script>

  <!--/radix_placeholder_distill-->
  <script src="meta-analysis_files/header-attrs-2.13/header-attrs.js"></script>
  <script src="meta-analysis_files/jquery-3.6.0/jquery-3.6.0.min.js"></script>
  <script src="meta-analysis_files/popper-2.6.0/popper.min.js"></script>
  <link href="meta-analysis_files/tippy-6.2.7/tippy.css" rel="stylesheet" />
  <link href="meta-analysis_files/tippy-6.2.7/tippy-light-border.css" rel="stylesheet" />
  <script src="meta-analysis_files/tippy-6.2.7/tippy.umd.min.js"></script>
  <script src="meta-analysis_files/anchor-4.2.2/anchor.min.js"></script>
  <script src="meta-analysis_files/bowser-1.9.3/bowser.min.js"></script>
  <script src="meta-analysis_files/webcomponents-2.0.0/webcomponents.js"></script>
  <script src="meta-analysis_files/distill-2.2.21/template.v2.js"></script>
  <!--radix_placeholder_site_in_header-->
  <!--/radix_placeholder_site_in_header-->


</head>

<body>

<!--radix_placeholder_front_matter-->

<script id="distill-front-matter" type="text/json">
{"title":"Effects of mycorrhizal association on litter decomposition in the forest ecosystems: A meta-analysis","description":"A short description of the post.","authors":[{"author":"Meenu Patil","authorURL":"https://www.researchgate.net/profile/Meenu-Patil","affiliation":"Panjab University, Chandigarh","affiliationURL":"https://puchd.ac.in","orcidID":"0000-0002-7664-7877"},{"author":"Abhishek Kumar","authorURL":"https://akumar.netlify.app","affiliation":"Panjab University, Chandigarh","affiliationURL":"https://puchd.ac.in","orcidID":"0000-0003-2252-7623"},{"author":"Pardeep Kumar","authorURL":"https://www.researchgate.net/profile/Pardeep-Kumar-22","affiliation":"Panjab University, Chandigarh","affiliationURL":"https://puchd.ac.in","orcidID":""},{"author":"Anand Narain Singh","authorURL":"https://www.researchgate.net/profile/Anand-Singh-15","affiliation":"Panjab University, Chandigarh","affiliationURL":"https://puchd.ac.in","orcidID":"0000-0002-0148-8680"}],"publishedDate":"2021-10-22T00:00:00.000+05:30","citationText":"Patil, et al., 2021"}
</script>

<!--/radix_placeholder_front_matter-->
<!--radix_placeholder_navigation_before_body-->
<!--/radix_placeholder_navigation_before_body-->
<!--radix_placeholder_site_before_body-->
<!--/radix_placeholder_site_before_body-->

<div class="d-title">
<h1>Effects of mycorrhizal association on litter decomposition in the forest ecosystems: A meta-analysis</h1>
<!--radix_placeholder_categories-->
<!--/radix_placeholder_categories-->
<p><p>A short description of the post.</p></p>
</div>

<div class="d-byline">
  Meenu Patil <a href="https://www.researchgate.net/profile/Meenu-Patil" class="uri">https://www.researchgate.net/profile/Meenu-Patil</a> (Panjab University, Chandigarh)<a href="https://puchd.ac.in" class="uri">https://puchd.ac.in</a>
  
,   Abhishek Kumar <a href="https://akumar.netlify.app" class="uri">https://akumar.netlify.app</a> (Panjab University, Chandigarh)<a href="https://puchd.ac.in" class="uri">https://puchd.ac.in</a>
  
,   Pardeep Kumar <a href="https://www.researchgate.net/profile/Pardeep-Kumar-22" class="uri">https://www.researchgate.net/profile/Pardeep-Kumar-22</a> (Panjab University, Chandigarh)<a href="https://puchd.ac.in" class="uri">https://puchd.ac.in</a>
  
,   Anand Narain Singh <a href="https://www.researchgate.net/profile/Anand-Singh-15" class="uri">https://www.researchgate.net/profile/Anand-Singh-15</a> (Panjab University, Chandigarh)<a href="https://puchd.ac.in" class="uri">https://puchd.ac.in</a>
  
<br/>2021-10-22
</div>

<div class="d-article">
<div class="d-contents d-contents-float">
<nav class="l-text toc figcaption" id="TOC">
<h3>Contents</h3>
<ul>
<li><a href="#abstract">Abstract</a></li>
<li><a href="#introduction">Introduction</a></li>
<li><a href="#methodology">Methodology</a>
<ul>
<li><a href="#literature-search">Literature search</a></li>
<li><a href="#article-screening">Article screening</a></li>
<li><a href="#data-extraction">Data extraction</a></li>
<li><a href="#phylogenetic-data">Phylogenetic data</a></li>
<li><a href="#effect-size-and-variance">Effect size and variance</a></li>
<li><a href="#meta-analytic-model">Meta-analytic model</a></li>
<li><a href="#heterogeneity">Heterogeneity</a></li>
<li><a href="#publication-bias">Publication bias</a></li>
</ul></li>
<li><a href="#results">Results</a>
<ul>
<li><a href="#overview-of-dataset">Overview of dataset</a></li>
<li><a href="#overall-effect-estimate">Overall effect estimate</a></li>
<li><a href="#publication-bias-1">Publication bias</a></li>
</ul></li>
<li><a href="#discussion">Discussion</a></li>
<li><a href="#conclusion">Conclusion</a></li>
</ul>
</nav>
</div>
<blockquote>
<p>This is the original draft that has not been peer-reviewed yet.</p>
</blockquote>
<h2 id="abstract">Abstract</h2>
<p>The presence of mycorrhiza can either increase or decrease the litter decomposition rates either by facilitation or competition. However, empirical evidence about the magnitude and direction of such effects remained unclear. We conducted a formal meta-analysis of published peer-reviewed literature to address this knowledge gap. Surprisingly, a systematic literature search with pre-defined inclusion criteria identified only five comparable studies. Therefore, based on 14 estimates from these studies, we evaluated the effects of mycorrhizal association on litter decomposition rates for nine species. Our meta-analysis with random-effect model showed strong evidence (RR = 1.07, <em>Z</em> = 3.58, <em>p</em> &lt; 0.001) for increased litter decomposition rates in presence of mycorrhizal fungi. The variations in the effects were influenced by species identity, but no effects were observed due to phylogeny. Thus, we found support for the mycorrhizal-mediated acceleration of litter decomposition, which questions the generality of the ‘Gadgil effect.’ Overall our findings suggest that the inclusion of mycorrhizal effects can advance our understanding of nutrient cycling and its responses to global environmental change. However, we need more studies conducted for a longer duration and representative of diverse ecosystems and litter types to arrive at a more robust conclusion.</p>
<p><strong>Keywords:</strong> gadgil effect; ectomycorrhiza; arbuscular mycorrhiza; forest ecosystem; litter decomposition; meta-analysis</p>
<h2 id="introduction">Introduction</h2>
<p>Global environmental change is expected to substantially alter the biogeochemistry and nutrient cycling in terrestrial ecosystems <span class="citation" data-cites="Shu2019">(<a href="#ref-Shu2019" role="doc-biblioref">Shu et al. 2019</a>)</span>. Among terrestrial ecosystems, forest ecosystems acts as significant sink for soil organic matter and therefore, substantially influence the global nutrient cycling <span class="citation" data-cites="Pan2011">(<a href="#ref-Pan2011" role="doc-biblioref">Pan et al. 2011</a>)</span>. The cycling of nutrients is primarily regulated by the process of litter decomposition, which releases nutrients from plant biomass to the soils <span class="citation" data-cites="Shu2019 Patil2020azb">(<a href="#ref-Shu2019" role="doc-biblioref">Shu et al. 2019</a>; <a href="#ref-Patil2020azb" role="doc-biblioref">Patil et al. 2020b</a>)</span>. Much of the variations in the litter decomposition rates are explained by climate and litter quality <span class="citation" data-cites="Zhang2008 Cornwell2008 Patil2020srep">(<a href="#ref-Zhang2008" role="doc-biblioref">Zhang et al. 2008</a>; <a href="#ref-Cornwell2008" role="doc-biblioref">Cornwell et al. 2008</a>; <a href="#ref-Patil2020srep" role="doc-biblioref">Patil et al. 2020a</a>)</span>. However, the remaining proportion of variation is often attributed to the composition and activity of decomposer organisms <span class="citation" data-cites="McGuire2010 Sulman2017">(<a href="#ref-McGuire2010" role="doc-biblioref">McGuire and Treseder 2010</a>; <a href="#ref-Sulman2017" role="doc-biblioref">Sulman et al. 2017</a>)</span>. Among decomposer organisms, bacteria and fungi are two major group organisms involved in litter decomposition <span class="citation" data-cites="Schneider2012 Heijden2015">(<a href="#ref-Schneider2012" role="doc-biblioref">Schneider et al. 2012</a>; <a href="#ref-Heijden2015" role="doc-biblioref">Heijden et al. 2015</a>)</span>. Although the role of saprotrophic fungi is appreciated in litter decomposition, mycorrhizal fungi can also substantially influence litter decomposition because they often act as mediators of nutrients between plants and soil <span class="citation" data-cites="Finlay2005 Frey2019 Kumar2021">(<a href="#ref-Finlay2005" role="doc-biblioref">Finlay 2005</a>; <a href="#ref-Frey2019" role="doc-biblioref">Frey 2019</a>; <a href="#ref-Kumar2021" role="doc-biblioref">Kumar et al. 2021</a>)</span>. Among the mycorrhizal associations, arbuscular mycorrhizal fungi (AMF) and ectomycorrhizal fungi (EMF) are commonly associated with vascular plants <span class="citation" data-cites="Brundrett2018">(<a href="#ref-Brundrett2018" role="doc-biblioref">Brundrett and Tedersoo 2018</a>)</span>. While the AMF dominates the tropical forests, EMF associations are frequently observed in extra-tropical forests <span class="citation" data-cites="Steidinger2019">(<a href="#ref-Steidinger2019" role="doc-biblioref">Steidinger et al. 2019</a>)</span>. Thus, the effects of mycorrhizal fungi are expected to vary not only among mycorrhizal groups, but also across the plant species, ecosystems and environmental conditions.</p>
<p>Mycorrhizal fungi have been observed to decrease the litter decomposition rates <span class="citation" data-cites="Gadgil1971 Gadgil1975">(<a href="#ref-Gadgil1971" role="doc-biblioref">Gadgil and Gadgil 1971</a>; <a href="#ref-Gadgil1975" role="doc-biblioref">Gadgil and Gadgil 1975</a>)</span>, which is frequently referred as ‘Gadgil effect’ in the literature <span class="citation" data-cites="Gadgil1971 Fernandez2016">(<a href="#ref-Gadgil1971" role="doc-biblioref">Gadgil and Gadgil 1971</a>; <a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span>. The ‘Gadgil effect’ has been observed for both ectomycorrhizal <span class="citation" data-cites="Gadgil1971">(<a href="#ref-Gadgil1971" role="doc-biblioref">Gadgil and Gadgil 1971</a>)</span> and arbuscular mycorrhizal fungi <span class="citation" data-cites="Leifheit2015">(<a href="#ref-Leifheit2015" role="doc-biblioref">Leifheit et al. 2015</a>)</span>. The resource competition between saprotrophs (free living microbes) and symbionts (mycorrhizal fungi) have been suggested to retard the decomposition rates in presence of mycorrhizal fungi <span class="citation" data-cites="Gadgil1971 Brzostek2015">(<a href="#ref-Gadgil1971" role="doc-biblioref">Gadgil and Gadgil 1971</a>; <a href="#ref-Brzostek2015" role="doc-biblioref">Brzostek et al. 2015</a>)</span>. The specialized enzymatic suite and foraging behaviour of these mycorrhizal fungi have been attributed to their ability to outcompete the saprotrophic fungi or other detritivore community <span class="citation" data-cites="Averill2014 Talbot2015">(<a href="#ref-Averill2014" role="doc-biblioref">Averill et al. 2014</a>; <a href="#ref-Talbot2015" role="doc-biblioref">Talbot et al. 2015</a>)</span>. Further, a literature review <span class="citation" data-cites="Fernandez2016">(<a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span> has also ascribed Gadgil effect to nitrogen competition <span class="citation" data-cites="Orwin2011 Averill2014">(<a href="#ref-Orwin2011" role="doc-biblioref">Orwin et al. 2011</a>; <a href="#ref-Averill2014" role="doc-biblioref">Averill et al. 2014</a>)</span>, chemical inhibition or allelopathy <span class="citation" data-cites="Krywolap1964 Rasanayagam1992">(<a href="#ref-Krywolap1964" role="doc-biblioref">Krywolap et al. 1964</a>; <a href="#ref-Rasanayagam1992" role="doc-biblioref">Rasanayagam and Jeffries 1992</a>)</span>, myco-parasitism <span class="citation" data-cites="Kubicek2011">(<a href="#ref-Kubicek2011" role="doc-biblioref">Kubicek et al. 2011</a>)</span>, and altered moisture-regimes <span class="citation" data-cites="Koide2003">(<a href="#ref-Koide2003" role="doc-biblioref">Koide and Wu 2003</a>)</span>.</p>
<p>In contrast to Gadgil effect, mycorrhizal fungi also reported to enhance the litter decomposition rates <span class="citation" data-cites="Frey2019">(<a href="#ref-Frey2019" role="doc-biblioref">Frey 2019</a>)</span> through extracellular enzymatic degradation <span class="citation" data-cites="Talbot2015">(<a href="#ref-Talbot2015" role="doc-biblioref">Talbot et al. 2015</a>)</span>, oxidation via Fenton chemistry <span class="citation" data-cites="Beeck2018">(<a href="#ref-Beeck2018" role="doc-biblioref">Beeck et al. 2018</a>)</span>, and priming effects, i.e., stimulation of growth and activities of saprotrophic decomposers by providing plant-derived carbon sources <span class="citation" data-cites="Herman2012 Gorka2019">(<a href="#ref-Herman2012" role="doc-biblioref">Herman et al. 2012</a>; <a href="#ref-Gorka2019" role="doc-biblioref">Gorka et al. 2019</a>)</span>. The priming effects appear to be more widespread as mycorrhizal fungi have limited genetic capabilities to produce organic matter degrading enzymes <span class="citation" data-cites="Frey2019">(<a href="#ref-Frey2019" role="doc-biblioref">Frey 2019</a>)</span>. The availability of labile exudates together with mycorrhizal necromass stimulates the activity of free-living bacteria and saprotrophic fungi, which leads to higher decomposition rates <span class="citation" data-cites="Fernandez2016">(<a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span>. Further, the the differences in ecophysiology of mycorrhizal groups are expected to modulate their effects on litter decomposition rates <span class="citation" data-cites="Tedersoo2019">(<a href="#ref-Tedersoo2019" role="doc-biblioref">Tedersoo and Bahram 2019</a>)</span>. For instance, the capacities for enzymatic degradation of organic matter widely vary among mycorrhizal taxa and it appears that EMF taxa have more degrading capacities than the AMF taxa <span class="citation" data-cites="Frey2019">(<a href="#ref-Frey2019" role="doc-biblioref">Frey 2019</a>)</span>. Therefore, considering the type of mycorrhizal association can be important in understanding the role of mycorrhizal fungi in nutrient cycling.</p>
<p>Thus, available literature suggest that mycorrhizal fungi can either enhance the litter decomposition by providing plant-derived carbon-source <span class="citation" data-cites="Zhu1996 Shah2016 Gui2017 Sterkenburg2018">(<a href="#ref-Zhu1996" role="doc-biblioref">Zhu and Ehrenfeld 1996</a>; <a href="#ref-Shah2016" role="doc-biblioref">Shah et al. 2016</a>; <a href="#ref-Gui2017" role="doc-biblioref">Gui et al. 2017</a>; <a href="#ref-Sterkenburg2018" role="doc-biblioref">Sterkenburg et al. 2018</a>)</span> or suppress it by competing for the same resources with saprotrophic fungi, also known as ‘Gadgil effect’ <span class="citation" data-cites="Gadgil1971 Gadgil1975 Brzostek2015">(<a href="#ref-Gadgil1971" role="doc-biblioref">Gadgil and Gadgil 1971</a>; <a href="#ref-Gadgil1975" role="doc-biblioref">Gadgil and Gadgil 1975</a>; <a href="#ref-Brzostek2015" role="doc-biblioref">Brzostek et al. 2015</a>)</span>. Nevertheless, some studies found little or no effects of mycorrhizal association on litter decomposition <span class="citation" data-cites="Staaf1988">(<a href="#ref-Staaf1988" role="doc-biblioref">Staaf 1988</a>)</span>. Therefore, the effects of mycorrhizal association on litter decomposition remained equivocal and highly variable in magnitude and direction, suggesting more complex mechanisms and contexts than previously thought <span class="citation" data-cites="Fernandez2016">(<a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span>. Although few exemplary syntheses about the effects of mycorrhizal association on litter decomposition exist <span class="citation" data-cites="Averill2014 Fernandez2016">(<a href="#ref-Averill2014" role="doc-biblioref">Averill et al. 2014</a>; <a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span>, quantitative estimates and direction of such effects remained ambiguous. Further, none of the earlier syntheses conducted a formal meta-analysis and their general conclusions might be confounded by sampling error or other sources of variations. Therefore, our primary objective is to quantitatively synthesise the magnitude and direction of mycorrhizal effects on litter decomposition rates in forest ecosystems. Further, we can expect substantial differences in effect due to species identity and phylogenetic relatedness among them because each species may have different quality of litter which can influence the decomposition rates. In addition to this, we aim to identify the knowledge gaps and suggestions for future investigations. Specifically, we aim to address the following questions:</p>
<ol type="1">
<li>Do mycorrhizal presence affect rates of litter decomposition in forest ecosystems?</li>
<li>What is the magnitude and direction of mycorrhizal effects on litter decomposition rates?</li>
<li>Are these effects influenced by species or their phylogenetic relatedness?</li>
<li>Do these effects differ between angiosperms and gymnosperms?</li>
</ol>
<p>To accomplish this, we carried out a formal meta-analysis of available data from systematically identified comparable studies. In this meta-analysis, we included a forest plot showing mycorrhiza’s overall effect on litter decomposition, between-study heterogeneity, and a funnel plot for showing publication bias of studies. We explored different meta-analytic models and assessed the robustness of our estimates. Then, we discussed the results of all studies briefly. In the end, we draw conclusions of the mycorrhizal effect on litter decomposition based on the results of our meta-analysis.</p>
<h2 id="methodology">Methodology</h2>
<p>Our methodology broadly consisted of five phases, i.e., searching, screening, data extraction, statistical meta-analysis and sensitivity analysis. We followed the guidelines developed by Collaboration on Environmental Evidence <span class="citation" data-cites="Pullin2018">(<a href="#ref-Pullin2018" role="doc-biblioref">Pullin et al. 2018</a>)</span> to identify eligible studies using a systematic literature search and screening protocol. The protocol was developed by organising meetings and implemented in a similar manner as described earlier <span class="citation" data-cites="Kumar2022">(<a href="#ref-Kumar2022" role="doc-biblioref">Kumar et al. 2022</a>)</span>. Further, the quality and reproducibility of meta-analysis was ensured by adhering to previously recommended guidelines <span class="citation" data-cites="Koricheva2014 Nakagawa2017">(<a href="#ref-Koricheva2014" role="doc-biblioref">Koricheva and Gurevitch 2014</a>; <a href="#ref-Nakagawa2017" role="doc-biblioref">Nakagawa et al. 2017</a>)</span>.</p>
<h3 id="literature-search">Literature search</h3>
<p>We systematically searched Scopus (<a href="https://scopus.com/" class="uri">https://scopus.com/</a>) and Web of Science Core Collection (<a href="https://webofknowledge.com/" class="uri">https://webofknowledge.com/</a>) on 17-18 February 2021 with the following search string:</p>
<blockquote>
<p>(mycorrhiza* OR ectomycorrhiza* OR “saprotrophic fungi” OR plant-fung*) AND (litter OR “litter decay” OR (litter AND decompos*) OR decomposition OR “nutrient acquisition”) AND (forest).</p>
</blockquote>
<p>This search string was developed by combining previously identified groups of keywords (see Supplementary information Table S1). Additionally, Google Scholar (<a href="https://scholar.google.com/" class="uri">https://scholar.google.com/</a>) was searched on 18-Feb-2021 using “Mycorrhiza decomposition” and “Mycorrhiza nutrient acquisition.” Thus, we obtained 789 records from Scopus, 1,293 records from Web of Science Core Collection, and 14 records from Google Scholar. All these 2,096 records were exported as BibTex files for further processing (see details in Supplementary Information).</p>
<div class="layout-chunk" data-layout="l-body">

</div>
<h3 id="article-screening">Article screening</h3>
<p>Our article screening involved de-duplication, title and abstract screening, and full-text screening. We left 1,455 records after de-duplication, and 641 duplicates were removed (Figure <a href="#fig:fig1">1</a>). All these records were imported to freely available Mendeley reference management software for further screening. We manually screened titles and abstracts of each document following a pre-defined criterion (see Supplementary information). Based on the title and abstract screening, we excluded 1,265 articles and left with 190 articles (Figure <a href="#fig:fig1">1</a>). We could not retrieve the full-text of six articles (see Supplementary information Table S2) and, therefore, excluded these six articles. Next, the full text of each article was screened based on PICO inclusion and exclusion criteria (Supplementary information Table S3). Full-text screening resulted in the exclusion of 181 articles based on PICO criteria (Figure <a href="#fig:fig1">1</a>). Thus, we were finally left with only three eligible studies <span class="citation" data-cites="Entry1991 Zhu1996 Lang2021">(<a href="#ref-Entry1991" role="doc-biblioref">Entry et al. 1991</a>; <a href="#ref-Zhu1996" role="doc-biblioref">Zhu and Ehrenfeld 1996</a>; <a href="#ref-Lang2021" role="doc-biblioref">Lang et al. 2021</a>)</span>. We further employed snowballing method to identify related eligible studies. We could not identify any suitable related study based on references of selected studies (backward snowballing). However, two more eligible articles <span class="citation" data-cites="Mayor2006 Fernandez2020">(<a href="#ref-Mayor2006" role="doc-biblioref">Mayor and Henkel 2006</a>; <a href="#ref-Fernandez2020" role="doc-biblioref">Fernandez et al. 2020</a>)</span> were identified through forward snowballing (citation tracking). Thus, five studies were included after article screening and snowballing.</p>
<div class="layout-chunk" data-layout="l-body">
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig1"></span>
<img src="data/Fig1.svg" alt="Schematic flow diagram for literature search, screening and inclusion process [modified from @Haddaway2018]"  />
<p class="caption">
Figure 1: Schematic flow diagram for literature search, screening and inclusion process <span class="citation" data-cites="Haddaway2018">(modified from <a href="#ref-Haddaway2018" role="doc-biblioref">Haddaway et al. 2018</a>)</span>
</p>
</div>
</div>
<h3 id="data-extraction">Data extraction</h3>
<p>We needed data on litter decomposition rates in the presence and absence of mycorrhiza to test our hypothesis. The litter decomposition rate is often expressed as annual decay constant usually estimated from Olson’s single exponential model <span class="citation" data-cites="Olson1963">(<a href="#ref-Olson1963" role="doc-biblioref">Olson 1963</a>)</span>. Therefore, we aimed to extract the decay constant (<span class="math inline">\(\kappa\)</span>) and associated standard error (SE) or standard deviation (SD) for each study to establish consistency in our analysis. Unfortunately, only a single study <span class="citation" data-cites="Entry1991">(<a href="#ref-Entry1991" role="doc-biblioref">Entry et al. 1991</a>)</span> reported the data in required form. All other studies have represented litter decomposition rates as mass remaining at different time intervals <span class="citation" data-cites="Zhu1996 Mayor2006 Fernandez2020 Lang2021">(<a href="#ref-Zhu1996" role="doc-biblioref">Zhu and Ehrenfeld 1996</a>; <a href="#ref-Mayor2006" role="doc-biblioref">Mayor and Henkel 2006</a>; <a href="#ref-Fernandez2020" role="doc-biblioref">Fernandez et al. 2020</a>; <a href="#ref-Lang2021" role="doc-biblioref">Lang et al. 2021</a>)</span>. In such cases, we extracted each remaining mass value in triplicate (one mean value and two standard error of mean values) at each given time interval using the WebPlotDigitizer (<a href="https://automeris.io/WebPlotDigitizer/" class="uri">https://automeris.io/WebPlotDigitizer/</a>). Thus, the actual number of replicates of litterbags at each sampling time was not taken into account; instead, we considered each data point as a sample to obtain a more uniform and comparable measure of litter mass remaining. In case of any ambiguity in timings, we considered the time stated in the methodology section of the published article <span class="citation" data-cites="Lang2021">(<a href="#ref-Lang2021" role="doc-biblioref">Lang et al. 2021</a>)</span>.</p>
<div class="layout-chunk" data-layout="l-body">

</div>
<p>When the mass remaining data was available, the decay constant (<span class="math inline">\(\kappa\)</span>) and its associated standard error (SE) was estimated by fitting a linearised form of Olson’s single exponential decay model <span class="citation" data-cites="Olson1963">(<a href="#ref-Olson1963" role="doc-biblioref">Olson 1963</a>)</span> as represented in Equation <a href="#eq:olson">(1)</a>.</p>
<p><span class="math display" id="eq:olson">\[\begin{equation}
\log(M_t) = -\kappa~t + \log(M_0)
\tag{1}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(M_0\)</span> is the initial litter mass, <span class="math inline">\(M_t\)</span> is the mass remaining at time <span class="math inline">\(t\)</span>, and <span class="math inline">\(\kappa\)</span> is the decay constant. When a study reported data for the same species at different sub-sites <span class="citation" data-cites="Fernandez2020">(<a href="#ref-Fernandez2020" role="doc-biblioref">Fernandez et al. 2020</a>)</span> or different seasons <span class="citation" data-cites="Entry1991">(<a href="#ref-Entry1991" role="doc-biblioref">Entry et al. 1991</a>)</span>, we estimated or recorded the decay constant (<span class="math inline">\(\kappa\)</span>) and its associated standard error (SE) or standard deviation (SD) separately. In addition to this, we recorded the plant species, mycorrhizal types, forest ecosystem types, litter types, and geographic coordinates of study sites. Further, we also noted technical information about methodologies, including the number of study experiments, duration of the investigation, exclusion method for mycorrhiza, and the characteristics of litter and litter bags.</p>
<h3 id="phylogenetic-data">Phylogenetic data</h3>
<p>To test the influence of phylogenetic relatedness, we used the recently published GBOTB backbone of phylogeny for seed plants, which is based on data from <a href="https://www.ncbi.nlm.nih.gov/genbank/">GenBank</a> and <a href="https://tree.opentreeoflife.org/opentree/argus/opentree13.4@ott93302">Open Tree of Life</a> <span class="citation" data-cites="Smith2018">(<a href="#ref-Smith2018" role="doc-biblioref">Smith and Brown 2018</a>)</span>. It is considered as the largest dated phylogeny for 79,881 taxa of seed plants resolved at species level. We used the package <code>V.PhyloMaker</code> (version 0.1.0) to prepare a phylogenetic tree for the selected plant species <span class="citation" data-cites="Jin2019">(<a href="#ref-Jin2019" role="doc-biblioref">Jin and Qian 2019</a>)</span>. The phylogeny was generated from the <code>GBOTB.extended</code> mega-tree, which is a combination of two published phylogenies for plants <span class="citation" data-cites="Smith2018 Zanne2014">(<a href="#ref-Zanne2014" role="doc-biblioref">Zanne et al. 2014</a>; <a href="#ref-Smith2018" role="doc-biblioref">Smith and Brown 2018</a>)</span>. The nodes were built using the <code>build.nodes.1</code> function, which returns the most recent common ancestor of all the tips in the largest cluster of the genus, and define it as the basal node of the genus. Further, the phylogeny was generated under the <em>Scenario 3</em> for binding of new genus tip to the family branch as defined in earlier study <span class="citation" data-cites="Qian2016">(<a href="#ref-Qian2016" role="doc-biblioref">Qian and Jin 2016</a>)</span>.</p>
<div class="layout-chunk" data-layout="l-body">

</div>
<h3 id="effect-size-and-variance">Effect size and variance</h3>
<p>Various effect sizes are used to summarize the magnitude and direction of the relationship between two variables <span class="citation" data-cites="Cooper2009 Harrer2021">(<a href="#ref-Cooper2009" role="doc-biblioref">Cooper et al. 2009</a>; <a href="#ref-Harrer2021" role="doc-biblioref">Harrer et al. 2021</a>)</span>. Among available effect sizes, response ratios (also called as Ratio of Means) and standardised mean differences have been frequently used to compare the means between two groups (i.e., experimental and control) in ecology and evolution <span class="citation" data-cites="Nakagawa2012 Koricheva2014">(<a href="#ref-Nakagawa2012" role="doc-biblioref">Nakagawa and Santos 2012</a>; <a href="#ref-Koricheva2014" role="doc-biblioref">Koricheva and Gurevitch 2014</a>)</span>. The response ratio or ratio of means is recommended when the mean of control group is not very small as compared to experimental group and both groups have estimates of same sign either positive or negative <span class="citation" data-cites="Hedges1999 Lajeunesse2015">(<a href="#ref-Hedges1999" role="doc-biblioref">Hedges et al. 1999</a>; <a href="#ref-Lajeunesse2015" role="doc-biblioref">Lajeunesse 2015</a>)</span>. The log transformation of response ratio (<span class="math inline">\(\text{lnRR}\)</span>) makes the metric equally sensitive to changes in both numerator and denominator, apart from making the metric nearly normal. Thus, it measures the actual difference in responses scaled to mean in control groups. However, <span class="math inline">\(\text{lnRR}\)</span> is considered to be slightly biased towards null effects and usually estimate high heterogeneity than other effect size measures <span class="citation" data-cites="Lajeunesse2015">(<a href="#ref-Lajeunesse2015" role="doc-biblioref">Lajeunesse 2015</a>)</span>.</p>
<p>Since the rate of litter decomposition is influenced by time (i.e., greater weight loss during initial phase than the later phases), we decided to compare decay constant rather than the weight loss. The log transformed response ratio (<span class="math inline">\(\text{lnRR}\)</span>) and associated variance was calculated using the Equation <a href="#eq:lnrr">(2)</a>.</p>
<p><span class="math display" id="eq:lnrr">\[\begin{equation}
\text{lnRR} = \ln \left( \frac{\bar X_E}{\bar X_C} \right); \quad
\text{Var(lnRR)} = \frac {SD_E^2}{n_E \bar X_E^2} + \frac {SD_C^2}{n_C \bar X_C^2}
\tag{2}
\end{equation}\]</span></p>
<p>where, <span class="math inline">\(\bar X_E\)</span> and <span class="math inline">\(\bar X_C\)</span> represents the absolute litter decay constant (<span class="math inline">\(\kappa\)</span>) for experimental (presence of mycorrhiza) and control (absence of mycorrhiza) groups, respectively. The large-sample approximation was used to compute the sampling variances (Equation <a href="#eq:lnrr">(2)</a>) associated with the effect size <span class="math inline">\(\text{lnRR}\)</span> <span class="citation" data-cites="Hedges1999">(<a href="#ref-Hedges1999" role="doc-biblioref">Hedges et al. 1999</a>)</span>. The accuracy of <span class="math inline">\(\text{lnRR}\)</span> was ensured with with Geary’s test (Equation <a href="#eq:geary">(3)</a>) for both control and experimental group <span class="citation" data-cites="Lajeunesse2015">(<a href="#ref-Lajeunesse2015" role="doc-biblioref">Lajeunesse 2015</a>)</span>.</p>
<p><span class="math display" id="eq:geary">\[\begin{equation}
\frac{\bar X}{SD} \sqrt{N} \geq 3
\tag{3}
\end{equation}\]</span></p>
<div class="layout-chunk" data-layout="l-body">

</div>
<p>A positive value of response ratio (<span class="math inline">\(\text{lnRR} &gt; 0\)</span>) indicates faster whereas a negative value (<span class="math inline">\(\text{lnRR} &lt; 0\)</span>) means slower decomposition rates in presence of mycorrhiza, and a value close to zero (<span class="math inline">\(\text{lnRR} \approx 0\)</span>) indicates little or no effect. Both effect size and associated sampling variances were calculated using the function <code>escalc()</code> from the <code>metafor</code> package <span class="citation" data-cites="metafor2010">(<a href="#ref-metafor2010" role="doc-biblioref">Viechtbauer 2010</a>)</span>.</p>
<div class="layout-chunk" data-layout="l-body">

</div>
<h3 id="meta-analytic-model">Meta-analytic model</h3>
<p>The overall aim of meta-analyses is to combine the effects or observations from different studies. There are three main types of meta-analytic models i.e., fixed-effect model (FEM), random-effects model (REM), and multilevel model (MLM), which are commonly used to compute the overall effect estimate in meta-analysis. The most basic fixed-effect or common-effect model assumes a common overall mean for all studies. Since it assumes homogeneity between studies, it is far from the reality, at least for ecological studies. In contrary to the fixed effect model, the assumptions of random-effects model are more realistic as it considers between-study heterogeneity. This model assumes that there is not only one actual effect size but a distribution of true effect sizes <span class="citation" data-cites="Borenstein2010 Harrer2021">(<a href="#ref-Borenstein2010" role="doc-biblioref">Borenstein et al. 2010</a>; <a href="#ref-Harrer2021" role="doc-biblioref">Harrer et al. 2021</a>)</span>. According to this model, the observed effect size <span class="math inline">\(y_i\)</span> with variance <span class="math inline">\(v_i\)</span> deviates from the mean true effect size <span class="math inline">\(\mu\)</span> for a single study <span class="math inline">\(i\)</span> (with <span class="math inline">\(i = 1, \ldots , N_{studies}\)</span>) by two error terms <span class="math inline">\(u_i\)</span> (between-study heterogeneity) and <span class="math inline">\(e_i\)</span> (sampling error). Thus, the random effects model can be expressed as Equation <a href="#eq:rem">(4)</a></p>
<p><span class="math display" id="eq:rem">\[\begin{equation}
y_i = \mu + u_i + e_i \tag{4} \\
\textbf{u} \sim \mathcal{N}(0, \sigma_u^2 \textbf{I}_u) \\
\textbf{e} \sim \mathcal{N}(0, \textbf{V}) 
\end{equation}\]</span></p>
<p>where, <span class="math inline">\(u_i\)</span> is the random effect corresponding to the <span class="math inline">\(i^{th}\)</span> study, <span class="math inline">\(\textbf{u}\)</span> is a <span class="math inline">\(1 \times N_{studies}\)</span> column vector with the <span class="math inline">\(u_i\)</span> values (which are assumed to be normally distributed with mean <span class="math inline">\(0\)</span> and variance <span class="math inline">\(\sigma_u^2\)</span>) and <span class="math inline">\(\textbf{I}_u\)</span> is an <span class="math inline">\(N_{studies} \times N_{studies}\)</span> identity matrix, <span class="math inline">\(\textbf{e}\)</span> is a <span class="math inline">\(1 \times N_{studies}\)</span> column vector with the <span class="math inline">\(e_{i}\)</span> values (which are assumed to be normally distributed with mean <span class="math inline">\(0\)</span> and variance <span class="math inline">\(v_i\)</span>), <span class="math inline">\(0\)</span> is a column vector of zeros and <span class="math inline">\(\textbf{V}\)</span> is an <span class="math inline">\(N_{studies} \times N_{studies}\)</span> matrix with the <span class="math inline">\(v_i\)</span> values along the diagonal.</p>
<p>Although the random-effects model is commonly used in ecology, it did not take into consideration the non-independence among effect sizes <span class="citation" data-cites="Nakagawa2012">(<a href="#ref-Nakagawa2012" role="doc-biblioref">Nakagawa and Santos 2012</a>)</span>. The non-independence can arise when the effect sizes are shared by studies (i.e., multiple effect sizes from the same study). Further, phylogenetic relatedness among species can also introduce non-independence among the effect size estimates <span class="citation" data-cites="Cinar2021">(<a href="#ref-Cinar2021" role="doc-biblioref">Cinar et al. 2021</a>)</span>. This issue of non-independence can be addressed by using a multilevel (hierarchical) meta-analytic model, which considers a random effect at each level of variability in effect sizes. Among the several factors for non-independence of effect sizes, we specifically assessed the dependence due to studies, species and phylogenetic relatedness among species as described recently <span class="citation" data-cites="Cinar2021">(<a href="#ref-Cinar2021" role="doc-biblioref">Cinar et al. 2021</a>)</span>. We used a complex multilevel meta-analytic model mathematically expressed in Equation <a href="#eq:mlm">(5)</a></p>
<p><span class="math display" id="eq:mlm">\[\begin{equation}
y_{ijk} = \mu + u_{ij} + s_i + n_k + p_k + e_{ij} \tag{5}\\
\textbf{u} \sim \mathcal{N}(0, \sigma_u^2 \textbf{I}_u) \\
\textbf{s} \sim \mathcal{N}(0, \sigma_s^2 \textbf{I}_s) \\
\textbf{n} \sim \mathcal{N}(0, \sigma_n^2 \textbf{I}_n) \\
\textbf{p} \sim \mathcal{N}(0, \sigma_p^2 \textbf{A}) \\
\textbf{e} \sim \mathcal{N}(0, \textbf{V}) 
\end{equation}\]</span></p>
<p>where</p>
<ul>
<li><span class="math inline">\(y_{ijk}\)</span> is the <span class="math inline">\(j^{th}\)</span> effect (with <span class="math inline">\(j = 1, \ldots, N_i\)</span>, where <span class="math inline">\(N_i\)</span> is the number of effect sizes reported in the <span class="math inline">\(i^{th}\)</span> study) in the <span class="math inline">\(i^{th}\)</span> study (with <span class="math inline">\(i = 1, \ldots, N_{studies}\)</span>) for the <span class="math inline">\(k^{th}\)</span> species (with <span class="math inline">\(k = 1, \ldots, N_{species}\)</span>), and <span class="math inline">\(v_{ijk}\)</span> are corresponding sampling variances,</li>
<li><span class="math inline">\(\mu\)</span> is the overall meta-analytic mean,</li>
<li><span class="math inline">\(u_{ij}\)</span> is a random effect corresponding to the <span class="math inline">\(j^{th}\)</span> effect size in the <span class="math inline">\(i^{th}\)</span> study,</li>
<li><span class="math inline">\(s_i\)</span> is a study-specific random effect for <span class="math inline">\(i^{th}\)</span> study,</li>
<li><span class="math inline">\(n_k\)</span> is a species-specific random effect for <span class="math inline">\(k^{th}\)</span> species,</li>
<li><span class="math inline">\(p_k\)</span> denotes the phylogenetic random effect for the <span class="math inline">\(k^{th}\)</span> species,</li>
<li><span class="math inline">\(e_{ij}\)</span> is the sampling error or residual corresponding to the <span class="math inline">\(j^{th}\)</span> effect size in the <span class="math inline">\(i^{th}\)</span> study,</li>
<li><span class="math inline">\(\textbf{u}\)</span> is a <span class="math inline">\(1 \times N_{total}\)</span> column vector with the <span class="math inline">\(u_{ij}\)</span> values (which are assumed to be normally distributed with mean <span class="math inline">\(0\)</span> and within-study variance <span class="math inline">\(\sigma_u^2\)</span>), <span class="math inline">\(N_{total} = \sum_{i = 1}^{N_{studies}} N_i\)</span> represents the total number of the effect sizes, <span class="math inline">\(\textbf{I}_u\)</span> is an <span class="math inline">\(N_{total} \times N_{total}\)</span> identity matrix,</li>
<li><span class="math inline">\(\textbf{s}\)</span> is a <span class="math inline">\(1 \times N_{studies}\)</span> column vector with the <span class="math inline">\(s_i\)</span> values (which are assumed to be normally distributed with mean <span class="math inline">\(0\)</span> and between-study variance <span class="math inline">\(\sigma_s^2\)</span>), <span class="math inline">\(\textbf{I}_s\)</span> is an <span class="math inline">\(N_{studies} \times N_{studies}\)</span> identity matrix,</li>
<li><span class="math inline">\(\textbf{n}\)</span> is a <span class="math inline">\(1 \times N_{species}\)</span> column vector with the <span class="math inline">\(n_k\)</span> values (which are assumed to be normally distributed with mean <span class="math inline">\(0\)</span> and between-species variance <span class="math inline">\(\sigma_n^2\)</span>) and <span class="math inline">\(\textbf{I}_n\)</span> has dimensions <span class="math inline">\(N_{species} × N_{species}\)</span>,</li>
<li><span class="math inline">\(\textbf{p}\)</span> is a <span class="math inline">\(1 \times N_{species}\)</span> column vector with the <span class="math inline">\(p_k\)</span> values (which are assumed to follow a multivariate normal distribution with mean <span class="math inline">\(0\)</span> and variance–covariance matrix <span class="math inline">\(\sigma_p^2 \textbf{A}\)</span>, where <span class="math inline">\(\sigma_p^2\)</span> denotes between-species variance due to the phylogeny, and <span class="math inline">\(\textbf{A}\)</span> is the <span class="math inline">\(N_{species} \times N_{species}\)</span> phylogenetic correlation matrix), and</li>
<li><span class="math inline">\(\textbf{e}\)</span> is a <span class="math inline">\(1 \times N_{total}\)</span> column vector with the <span class="math inline">\(e_{ij}\)</span> values and <span class="math inline">\(\textbf{V}\)</span> is the corresponding (diagonal) variance–covariance matrix with dimensions <span class="math inline">\(N_{total} \times N_{total}\)</span>.</li>
</ul>
<p>The random-effects model was fitted using the <code>rma()</code> function whereas the complex multilevel meta-analytic model was fitted using the <code>rma.mv()</code> function from the <code>metafor</code> package <span class="citation" data-cites="metafor2010">(<a href="#ref-metafor2010" role="doc-biblioref">Viechtbauer 2010</a>)</span>. The amount of heterogeneity under the both models were estimated using the less biased restricted maximum likelihood (REML) estimator <span class="citation" data-cites="Viechtbauer2005">(<a href="#ref-Viechtbauer2005" role="doc-biblioref">Viechtbauer 2005</a>)</span>. The phylogenetic correlation matrix, denoted as <span class="math inline">\(\textbf{A}\)</span> in Equation <a href="#eq:mlm">(5)</a>, was computed under the Brownian model using the function <code>vcv()</code> from the <code>ape</code> package version 5.6.2 <span class="citation" data-cites="ape2019">(<a href="#ref-ape2019" role="doc-biblioref">Paradis and Schliep 2019</a>)</span>. The branch lengths for each species was computed with the Grafen’s method <span class="citation" data-cites="Grafen1989">(<a href="#ref-Grafen1989" role="doc-biblioref">Grafen 1989</a>)</span> using the function <code>compute.brlen()</code> from the same package <span class="citation" data-cites="ape2019">(<a href="#ref-ape2019" role="doc-biblioref">Paradis and Schliep 2019</a>)</span>.</p>
<div class="layout-chunk" data-layout="l-body">

</div>
<h3 id="heterogeneity">Heterogeneity</h3>
<p>The heterogeneity reflects the variations in true effect sizes, which is not accounted by the sampling error variance within the meta-analysis. Cochran’s <span class="math inline">\(Q\)</span> is traditionally used in meta-analysis to assess the heterogeneity in meta-analysis <span class="citation" data-cites="Cochran1954">(<a href="#ref-Cochran1954" role="doc-biblioref">Cochran 1954</a>)</span>. It is the inverse variance weighted sum of squares, which can be mathematically represented by Equation <a href="#eq:cochran">(6)</a></p>
<p><span class="math display" id="eq:cochran">\[\begin{equation}
Q = \sum_{i=1}^{N_{studies}} w_i (y_i - \mu)^2; 
\quad w_i = \frac{1}{v_i}
\tag{6}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(y_i\)</span> is the observed effect for each study <span class="math inline">\(i^{th}\)</span>; <span class="math inline">\(\mu\)</span> is the overall summary effect, <span class="math inline">\(w_i\)</span> is the weight defined as inverse of variance (<span class="math inline">\(1/v_i\)</span>) for <span class="math inline">\(i^{th}\)</span> study.</p>
<p>Although Cochran’s <span class="math inline">\(Q\)</span> has been widely used to test heterogeneity in meta-analyses <span class="citation" data-cites="Nakagawa2012 Koricheva2014">(<a href="#ref-Nakagawa2012" role="doc-biblioref">Nakagawa and Santos 2012</a>; <a href="#ref-Koricheva2014" role="doc-biblioref">Koricheva and Gurevitch 2014</a>)</span>, it is sensitive to size and precision of studies included meta-analysis <span class="citation" data-cites="Borenstein2009 Harrer2021">(<a href="#ref-Borenstein2009" role="doc-biblioref">Borenstein et al. 2009</a>; <a href="#ref-Harrer2021" role="doc-biblioref">Harrer et al. 2021</a>)</span>. Higgins and Thompson’s <span class="math inline">\(I^2\)</span> statistic presents the more standardized form to estimate the between-study heterogeneity. It is defined as the percentage of variability in the effect size that is not caused by sampling error <span class="citation" data-cites="Higgins2002">(<a href="#ref-Higgins2002" role="doc-biblioref">Higgins and Thompson 2002</a>)</span>. Similarly, the <span class="math inline">\(H^2\)</span> statistic is used to describe the variance in observed effect sizes due to sampling error <span class="citation" data-cites="Higgins2002">(<a href="#ref-Higgins2002" role="doc-biblioref">Higgins and Thompson 2002</a>)</span>. It is estimated as the ratio of observed variation to the expected variance due to sampling error. Although both <span class="math inline">\(I^2\)</span> and <span class="math inline">\(H^2\)</span> are measured based on the Cochran’s <span class="math inline">\(Q\)</span>, the Higgins and Thompson’s <span class="math inline">\(I^2\)</span> and <span class="math inline">\(H^2\)</span> were calculated based on more general definitions using the Equation <a href="#eq:i2rem">(7)</a>.</p>
<p><span class="math display" id="eq:s2e" id="eq:h2rem" id="eq:i2rem">\[\begin{align}
I^2 &amp;= \frac{\sigma_u^2}{\sigma_u^2 + \sigma_e^2} \times 100 \tag{7} \\
\\
H^2 &amp;= \frac{\sigma_u^2 + \sigma_e^2}{\sigma_e^2} \tag{8} \\
\\
\sigma_e^2 &amp;=\frac{(N_{studies}-1)\sum w_i}{(\sum w_i)^2 - \sum w_i^2} \tag{9}
\end{align}\]</span></p>
<p>where <span class="math inline">\(\sigma_u^2\)</span> is the between-study variance (also referred as <span class="math inline">\(\tau^2\)</span>), <span class="math inline">\(\sigma_e^2\)</span> is the typical within-study variance, <span class="math inline">\(N_{studies}\)</span> is the total number of studies, <span class="math inline">\(w_i\)</span> is the weight defined as inverse of variance (<span class="math inline">\(1/v_i\)</span>) for <span class="math inline">\(i^{th}\)</span> study. We also included a prediction interval for the true overall summary effect (<span class="math inline">\(\mu\)</span>) because high heterogeneity is expected for ecological studies <span class="citation" data-cites="Senior2016">(<a href="#ref-Senior2016" role="doc-biblioref">Senior et al. 2016</a>)</span>.</p>
<p>Since <span class="math inline">\(Q\)</span> and <span class="math inline">\(I^2\)</span> are not designed to assess between-study heterogeneity for multilevel models <span class="citation" data-cites="Nakagawa2012">(<a href="#ref-Nakagawa2012" role="doc-biblioref">Nakagawa and Santos 2012</a>)</span>, we computed heterogeneity <span class="math inline">\(I^2\)</span> for our multilevel model as suggested earlier <span class="citation" data-cites="Nakagawa2012">(<a href="#ref-Nakagawa2012" role="doc-biblioref">Nakagawa and Santos 2012</a>)</span>. Following the previous mathematical notations, the within-study heterogeneity <span class="math inline">\(I_u^2\)</span> can be represented as Equation <a href="#eq:i2mlm">(10)</a> whereas <span class="math inline">\(H^2\)</span> or phylogenetic heritability can be represented by Equation <a href="#eq:h2mlm">(11)</a>. The values of <span class="math inline">\(H^2\)</span> can range from 0 to 1, which corresponds to no phylogenetic relatedness to exact proportional to phylogenetic relatedness among effect sizes.</p>
<p><span class="math display" id="eq:h2mlm" id="eq:i2mlm">\[\begin{align}
I_u^2 &amp;= \frac {\sigma_u^2} {\sigma_u^2 + \sigma_s^2 + \sigma_n^2 + \sigma_p^2 + \sigma_e^2} \tag{10} \\
\\
H^2 &amp;= \frac{\sigma_p^2}{\sigma_u^2 + \sigma_s^2 + \sigma_n^2 + \sigma_p^2} \tag{11}
\end{align}\]</span></p>
<div class="layout-chunk" data-layout="l-body">

</div>
<p><strong>Meta-regression:</strong> The heterogeneity in the observed outcomes can be explained using a set of predictors or moderators in meta-analysis. Such effect size partitioning among various variables is sometimes known as subgroup analysis. In absence of potentially better predictors, we explored species identity and clade (angiosperm or gymnosperm) as predictors since they can act as a surrogate for species level traits including the litter quality. We fitted mixed-effects models to assess the clade-specific or species-specific variations in the effects.</p>
<p><strong>Outlier and influential analysis:</strong> The overall pooled effect size can be heavily influenced by one or more outcomes (effect size), so that our estimated effect size is not robust. We assessed the outliers and influential cases using the studentised residuals and Cook’s distances for our random effects model <span class="citation" data-cites="Viechtbauer2010">(<a href="#ref-Viechtbauer2010" role="doc-biblioref">Viechtbauer and Cheung 2010</a>)</span>. Outcomes with a studentised residual larger than the <span class="math inline">\(100 \times (1 − 0.05 / (2 \times N_{total}))^{th}\)</span> percentile of a standard normal distribution are considered potential outliers (i.e., using a Bonferroni correction with two-sided <span class="math inline">\(\alpha = 0.05\)</span> for <span class="math inline">\(N_{total}\)</span> outcomes included in the meta-analysis). Similarly, outcomes with a Cook’s distance larger than the median plus six times the interquartile range of the Cook’s distances are considered to be influential.</p>
<h3 id="publication-bias">Publication bias</h3>
<p>The publication bias in the average effect size was assessed using the classical funnel plot. It is a scatter plot of the observed effect size for each study on the <em>x</em>-axis against a measure of the standard error on the <em>y</em>-axis of each study. In the absence of any publication bias, effect size distribution should roughly follow the funnel shape in the plot. We conducted the rank correlation test <span class="citation" data-cites="Begg1994">(<a href="#ref-Begg1994" role="doc-biblioref">Begg and Mazumdar 1994</a>)</span> and the regression test <span class="citation" data-cites="Sterne2006">(<a href="#ref-Sterne2006" role="doc-biblioref">Sterne and Egger 2006</a>)</span> to assess the asymmetry in the funnel plot. The regression was carried using the standard error of the observed outcomes as predictor.</p>
<p>Further, we computed the fail-safe numbers to assess the robustness of estimated average effect. Fail-safe numbers estimate the number of non-significant and or unpublished studies which would reduce the significance of overall average estimate if included in the meta-analysis. We used the unweighted Rosenthal’s <span class="citation" data-cites="Rosenthal1979">(<a href="#ref-Rosenthal1979" role="doc-biblioref">Rosenthal 1979</a>)</span> and weighted Rosenberg’s <span class="citation" data-cites="Rosenberg2005">(<a href="#ref-Rosenberg2005" role="doc-biblioref">Rosenberg 2005</a>)</span> approaches to estimate the fail-safe numbers.</p>
<div class="layout-chunk" data-layout="l-body">

</div>
<p>All analyses were performed in <code>R</code> programming language and statistical environment version 4.1.0 <span class="citation" data-cites="R2021">(<a href="#ref-R2021" role="doc-biblioref">R Core Team 2021</a>)</span>. The <code>R</code> package <code>synthesisr</code> version 0.3.0 <span class="citation" data-cites="Westgate2020">(<a href="#ref-Westgate2020" role="doc-biblioref">Westgate and Grames 2020</a>)</span> was used for the de-duplication of articles, package <code>metafor</code> version 3.1.43 <span class="citation" data-cites="metafor2010">(<a href="#ref-metafor2010" role="doc-biblioref">Viechtbauer 2010</a>)</span> for statistical meta-analysis, package <code>ggtree</code> version 3.0.4 used for visualisation of phylogenetic tree <span class="citation" data-cites="ggtree2016">(<a href="#ref-ggtree2016" role="doc-biblioref">Yu et al. 2016</a>)</span> and package <code>tidyverse</code> version 1.3.1 <span class="citation" data-cites="Wickham2019">(<a href="#ref-Wickham2019" role="doc-biblioref">Wickham et al. 2019</a>)</span> for general data manipulation and visualization.</p>
<h2 id="results">Results</h2>
<h3 id="overview-of-dataset">Overview of dataset</h3>
<p>Our systematic literature search and screening found five eligible studies for the present analysis (Figure <a href="#fig:fig1">1</a>). Studies in our dataset differed in numerous ways, including experimental duration, sampling frequency, initial litter mass, litter species, mycorrhiza type, and forest type (Table <a href="#tab:tab1">1</a>).</p>
<div class="layout-chunk" data-layout="l-body">
<div id="tfxxwmajcy" style="overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>html {
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', 'Fira Sans', 'Droid Sans', Arial, sans-serif;
}

#tfxxwmajcy .gt_table {
  display: table;
  border-collapse: collapse;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: 100%;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 0px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#tfxxwmajcy .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#tfxxwmajcy .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#tfxxwmajcy .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 0;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#tfxxwmajcy .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#tfxxwmajcy .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #000000;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#tfxxwmajcy .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#tfxxwmajcy .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#tfxxwmajcy .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#tfxxwmajcy .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#tfxxwmajcy .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#tfxxwmajcy .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
}

#tfxxwmajcy .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#tfxxwmajcy .gt_from_md > :first-child {
  margin-top: 0;
}

#tfxxwmajcy .gt_from_md > :last-child {
  margin-bottom: 0;
}

#tfxxwmajcy .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#tfxxwmajcy .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#tfxxwmajcy .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#tfxxwmajcy .gt_row_group_first td {
  border-top-width: 2px;
}

#tfxxwmajcy .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#tfxxwmajcy .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#tfxxwmajcy .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#tfxxwmajcy .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#tfxxwmajcy .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#tfxxwmajcy .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#tfxxwmajcy .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#tfxxwmajcy .gt_table_body {
  border-top-style: solid;
  border-top-width: 3px;
  border-top-color: #000000;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #000000;
}

#tfxxwmajcy .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#tfxxwmajcy .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-left: 4px;
  padding-right: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#tfxxwmajcy .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#tfxxwmajcy .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#tfxxwmajcy .gt_left {
  text-align: left;
}

#tfxxwmajcy .gt_center {
  text-align: center;
}

#tfxxwmajcy .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#tfxxwmajcy .gt_font_normal {
  font-weight: normal;
}

#tfxxwmajcy .gt_font_bold {
  font-weight: bold;
}

#tfxxwmajcy .gt_font_italic {
  font-style: italic;
}

#tfxxwmajcy .gt_super {
  font-size: 65%;
}

#tfxxwmajcy .gt_footnote_marks {
  font-style: italic;
  font-weight: normal;
  font-size: 75%;
  vertical-align: 0.4em;
}

#tfxxwmajcy .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#tfxxwmajcy .gt_slash_mark {
  font-size: 0.7em;
  line-height: 0.7em;
  vertical-align: 0.15em;
}

#tfxxwmajcy .gt_fraction_numerator {
  font-size: 0.6em;
  line-height: 0.6em;
  vertical-align: 0.45em;
}

#tfxxwmajcy .gt_fraction_denominator {
  font-size: 0.6em;
  line-height: 0.6em;
  vertical-align: -0.05em;
}
</style>
<table class="gt_table">
  <caption><span id="tab:tab1">Table 1: </span>Overview of selected studies included in the meta-analysis</caption>
  
  <thead class="gt_col_headings">
    <tr>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1">Author</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1">Year</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1">Experiments</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1">Species</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1">Sampling frequency</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1">Initial mass<sup class="gt_footnote_marks">1</sup></th>
      <th class="gt_col_heading gt_columns_bottom_border gt_right" rowspan="1" colspan="1">Mesh size<sup class="gt_footnote_marks">2</sup></th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1">Litter type</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1">Mycorrhiza type</th>
      <th class="gt_col_heading gt_columns_bottom_border gt_left" rowspan="1" colspan="1">Forest type</th>
    </tr>
  </thead>
  <tbody class="gt_table_body">
    <tr><td class="gt_row gt_left">Entry et al</td>
<td class="gt_row gt_right">1991</td>
<td class="gt_row gt_right">4</td>
<td class="gt_row gt_right">1</td>
<td class="gt_row gt_right">1</td>
<td class="gt_row gt_right">5.00</td>
<td class="gt_row gt_right">1</td>
<td class="gt_row gt_left">Leaf</td>
<td class="gt_row gt_left">EMF</td>
<td class="gt_row gt_left">Temperate</td></tr>
    <tr><td class="gt_row gt_left">Zhu &amp; Ehrenfeld</td>
<td class="gt_row gt_right">1996</td>
<td class="gt_row gt_right">1</td>
<td class="gt_row gt_right">1</td>
<td class="gt_row gt_right">4</td>
<td class="gt_row gt_right">1.63</td>
<td class="gt_row gt_right">1</td>
<td class="gt_row gt_left">Leaf</td>
<td class="gt_row gt_left">EMF</td>
<td class="gt_row gt_left">Temperate</td></tr>
    <tr><td class="gt_row gt_left">Mayor &amp; Henkel</td>
<td class="gt_row gt_right">2006</td>
<td class="gt_row gt_right">1</td>
<td class="gt_row gt_right">1</td>
<td class="gt_row gt_right">3</td>
<td class="gt_row gt_right">17.00</td>
<td class="gt_row gt_right">1</td>
<td class="gt_row gt_left">Leaf</td>
<td class="gt_row gt_left">EMF</td>
<td class="gt_row gt_left">Tropical</td></tr>
    <tr><td class="gt_row gt_left">Fernandez et al</td>
<td class="gt_row gt_right">2020</td>
<td class="gt_row gt_right">4</td>
<td class="gt_row gt_right">2</td>
<td class="gt_row gt_right">3</td>
<td class="gt_row gt_right">2.00</td>
<td class="gt_row gt_right">2</td>
<td class="gt_row gt_left">Leaf</td>
<td class="gt_row gt_left">EMF &amp; AMF</td>
<td class="gt_row gt_left">Temperate</td></tr>
    <tr><td class="gt_row gt_left">Lang et al</td>
<td class="gt_row gt_right">2021</td>
<td class="gt_row gt_right">4</td>
<td class="gt_row gt_right">4</td>
<td class="gt_row gt_right">3</td>
<td class="gt_row gt_right">1.00</td>
<td class="gt_row gt_right">2</td>
<td class="gt_row gt_left">Leaf</td>
<td class="gt_row gt_left">EMF</td>
<td class="gt_row gt_left">Temperate</td></tr>
  </tbody>
  <tfoot class="gt_sourcenotes">
    <tr>
      <td class="gt_sourcenote" colspan="10">EMF = ectomycorrhizal fungi, AMF = arbuscular mycorrhizal fungi</td>
    </tr>
  </tfoot>
  <tfoot class="gt_footnotes">
    <tr>
      <td class="gt_footnote" colspan="10"><sup class="gt_footnote_marks">1</sup> in grams</td>
    </tr>
    <tr>
      <td class="gt_footnote" colspan="10"><sup class="gt_footnote_marks">2</sup> in mm</td>
    </tr>
  </tfoot>
</table>
</div>
</div>
<p>These studies conducted total 14 experiments for nine different species. The leaf litter of nine plant species (<em>Acer saccharum</em> Marshall, <em>Betula alleghaniensis</em> Britton, <em>Dicymbe corymbosa</em> Benth., <em>Fagus grandifolia</em> Ehrh., <em>Fraxinus americana</em> L., <em>Pinus rigida</em> Mill., <em>Pinus strobus</em> L., <em>Pseudotsuga menziesii</em> (Mirbel) Franco <em>Quercus ellipsoidalis</em> E. J. Hill) was used during the experiments (Figure <a href="#fig:fig2">2</a>a). The duration of experiments is also differed among studies and ranged from one year to 2.4 years. The initial leaf litter mass varied from 1.0 g to 17.0 g, and three studies used up to 2.0 g of leaf litter, whereas two studies used more than 2.0 g of leaf litter <span class="citation" data-cites="Entry1991 Mayor2006">(<a href="#ref-Entry1991" role="doc-biblioref">Entry et al. 1991</a>; <a href="#ref-Mayor2006" role="doc-biblioref">Mayor and Henkel 2006</a>)</span>. Further, most of the studies involved ectomycorrhizal fungi <span class="citation" data-cites="Entry1991 Zhu1996 Mayor2006 Fernandez2020">(<a href="#ref-Entry1991" role="doc-biblioref">Entry et al. 1991</a>; <a href="#ref-Zhu1996" role="doc-biblioref">Zhu and Ehrenfeld 1996</a>; <a href="#ref-Mayor2006" role="doc-biblioref">Mayor and Henkel 2006</a>; <a href="#ref-Fernandez2020" role="doc-biblioref">Fernandez et al. 2020</a>)</span>, while one study showed the involvement of both ectomycorrhizal and arbuscular mycorrhizal fungi <span class="citation" data-cites="Lang2021">(<a href="#ref-Lang2021" role="doc-biblioref">Lang et al. 2021</a>)</span>. All the studies assured mycorrhizal exclusion by trenching to 30 cm soil depth.</p>
<p>Further, our dataset revealed that most studies were conducted in the temperate deciduous forest of North America (n = 4), and only one study was conducted in the tropical forest of Guyana of South America’s North Atlantic Coast (Figure <a href="#fig:fig2">2</a>b). The earliest study included was from 1991 <span class="citation" data-cites="Entry1991">(<a href="#ref-Entry1991" role="doc-biblioref">Entry et al. 1991</a>)</span>, whereas the latest study was published in 2021 <span class="citation" data-cites="Lang2021">(<a href="#ref-Lang2021" role="doc-biblioref">Lang et al. 2021</a>)</span>.</p>
<div class="layout-chunk" data-layout="l-body">
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig2"></span>
<img src="meta-analysis_files/figure-html5/fig2-1.svg" alt="(a) Phylogenetic relationship among the species used for litter decomposition. The symbols represents the studies to which a species belong. (b) Geographical distribution of studies included in our dataset" width="960" />
<p class="caption">
Figure 2: (a) Phylogenetic relationship among the species used for litter decomposition. The symbols represents the studies to which a species belong. (b) Geographical distribution of studies included in our dataset
</p>
</div>
</div>
<h3 id="overall-effect-estimate">Overall effect estimate</h3>
<p>A total of 14 estimates of litter decomposition (<span class="math inline">\(\kappa\)</span>) were included in the meta-analysis (Figure <a href="#fig:fig3">3</a>). The observed response ratios (<span class="math inline">\(\text{RR}\)</span>) ranged from 0.74 to 1.58, with the majority of estimates being positive (71%). Thus, the decomposition rate is also expected to be higher in the presence of mycorrhiza on average (Figure <a href="#fig:fig3">3</a>). Further, a strong positive relationship was observed between annual decay constants for the presence and absence of mycorrhiza (<em>R<sup>2</sup></em> = 0.94, <em>p</em> &lt; 0.001), indicating that our dataset does not include extreme effect estimates.</p>
<div class="layout-chunk" data-layout="l-body">
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig3"></span>
<img src="meta-analysis_files/figure-html5/fig3-1.svg" alt="Effects of mycorrhiza on annual litter decay constant ($\kappa$). Each point represents a single comparison of $\kappa$. The color of each point denotes the change in $\kappa$ due to the presence of mycorrhiza. Values above and below the 1:1 line represents the stimulation and inhibition of litter decomposition rate in presence of mycorrhiza, respectively. The values falling on the 1:1 line represent the neutral or no effect" width="672" />
<p class="caption">
Figure 3: Effects of mycorrhiza on annual litter decay constant (<span class="math inline">\(\kappa\)</span>). Each point represents a single comparison of <span class="math inline">\(\kappa\)</span>. The color of each point denotes the change in <span class="math inline">\(\kappa\)</span> due to the presence of mycorrhiza. Values above and below the 1:1 line represents the stimulation and inhibition of litter decomposition rate in presence of mycorrhiza, respectively. The values falling on the 1:1 line represent the neutral or no effect
</p>
</div>
</div>
<p>Consistent with our expectations, both meta-analytic models also suggested an overall positive effect of mycorrhizal fungi on litter decomposition rates. The overall average effect estimate (<span class="math inline">\(\hat \mu\)</span>) was 1.07 (<em>95% CI</em>: 1.03, 1.11) for the random effects model whereas the same was 1.08 (<em>95% CI</em>: 1, 1.18) for the multilevel model. The wider confidence intervals are expected in the latter model as it included additional sources of variations (number of levels, i.e., id, study, species). In ecological sense, these models indicated that on average the annual decay constant for leaf litter was 7-8% higher in presence of mycorrhizal fungi (Figure <a href="#fig:fig4">4</a>). The random-effects model found strong statistical evidence (<em>Z</em> = 3.58, <em>p</em> &lt; 0.001) whereas the multilevel model showed moderate statistical evidence (<em>Z</em> = 1.96, <em>p</em> = 0.05) for the accelerated decomposition rates. This positive effect is also evident in the forest plot, where the right-side position of the diamond demonstrates that litter decomposition rate increased in the presence of mycorrhizal fungi (Figure <a href="#fig:fig4">4</a>).</p>
<div class="layout-chunk" data-layout="l-body">
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig4"></span>
<img src="meta-analysis_files/figure-html5/fig4-1.svg" alt="Forest plot showing the observed effects of mycorrhizal association on litter decomposition rates for each experiment (n = 14). The size of each square is proportional to the weight of each study, and the associated bars represent the 95% confidence interval associated with individual effect size. The diamonds at the bottom represent the overall average effect estimated from meta-analytic models. The length of diamond denotes the 95% confidence intervals whereas associated dotted lines indicate the 95% prediction intervals" width="768" />
<p class="caption">
Figure 4: Forest plot showing the observed effects of mycorrhizal association on litter decomposition rates for each experiment (n = 14). The size of each square is proportional to the weight of each study, and the associated bars represent the 95% confidence interval associated with individual effect size. The diamonds at the bottom represent the overall average effect estimated from meta-analytic models. The length of diamond denotes the 95% confidence intervals whereas associated dotted lines indicate the 95% prediction intervals
</p>
</div>
</div>
<p>The Cochran’s <span class="math inline">\(Q\)</span> test for heterogeneity revealed significant variations (<span class="math inline">\(Q\)</span> = 259.45, <em>df</em> = 13, <em>p</em> &lt; 0.001) in true outcomes. Further, very high value of Higgins &amp; Thompson’s statistic (<span class="math inline">\(I^2\)</span> = 97.85%; <em>95% CI</em>: 97.77, 99.94) suggest that most of the observed variations captured the real differences in true effects (<span class="math inline">\(\tau^2\)</span> = 0.002). Similarly, the within-study variations in observed estimates were larger than the sampling error as indicated by <span class="math inline">\(H^2\)</span> statistic (<span class="math inline">\(H^2\)</span> = 46.43).</p>
<div class="layout-chunk" data-layout="l-body">

</div>
<p>Consistent with random-effects model, the multilevel model also indicated very high heterogeneity (<span class="math inline">\(I_t^2\)</span> = 99.5%) in true outcomes. The main advantage of multilevel model is that it allows us to estimate the heterogeneity in true outcomes at different levels of meta-analytic model. This model suggested that the heterogeneity due to species-specific effects was very large (<span class="math inline">\(\sigma_n^2\)</span> = 0.007; <span class="math inline">\(I_n^2\)</span> = 84.32%) followed by the within-study heterogeneity (<span class="math inline">\(\sigma_u^2\)</span> = 0.001; <span class="math inline">\(I_u^2\)</span> = 15.22%). Interestingly, neither between-study heterogeneity (<span class="math inline">\(\sigma_s^2\)</span> = 0; <span class="math inline">\(I_s^2\)</span> = 0) nor phylogeny-specific heterogeneity (<span class="math inline">\(\sigma_p^2\)</span> = 0; <span class="math inline">\(I_p^2\)</span> = 0) was detected by our multilevel model. Since no phylogenetic effects were observed, the phylogenetic heritability (<span class="math inline">\(H^2\)</span> = 0) was also absent.</p>
<p>Given the high heterogeneity in true outcomes, the 95% prediction interval ranged from 0.97 to 1.17 as computed by the random effects model. However, the multilevel model suggested a much wider 95% prediction interval ranging from as low as 0.89 to as high as 1.33. Since the predicted values of average estimate included values both lesser and greater than one, we can expect slower decomposition rates in some experiments, though the presence of mycorrhizal fungi enhanced the litter decomposition on average (Figure <a href="#fig:fig4">4</a>).</p>
<div class="layout-chunk" data-layout="l-body">

</div>
<p>The meta-regression model with clade (Angiosperm vs Gymnosperm) as categorical moderator suggested that experiments using litter of gymnosperm species obtained on average smaller effect (<span class="math inline">\(\beta_1\)</span> = -0.03, <em>SE</em> = 0.05) as compared to angiosperms, though this difference was not statistically significant (<span class="math inline">\(Q_M\)</span> = 0.31, <em>df</em> = 1, <em>p</em> = 0.578). Thus, inclusion of clade as moderator variable did not explained any variation in true effects. While the mixed-effects model with species as moderator reflected the variations of effect estimates among species (Figure <a href="#fig:fig5">5</a>). This meta-regression model indicated that at least one species had significantly different (higher or lower) response ratio than the response ratio for the reference level (<em>Acer saccharum</em> in our case). Thus, this model provided moderate statistical evidence that some amount of variations in true effects are indeed due to species (<span class="math inline">\(Q_M\)</span> = 16.96, <em>df</em> = 8, <em>p</em> = 0.031). It appeared that about 29% of the total variations in the true outcomes were due to species. Although species explained some proportion of the total variations, there still exist variations in true effects that are not attributable to species. These unexplained variations are evident from the test for residual heterogeneity (<span class="math inline">\(Q\)</span> = 239.52, <em>df</em> = 5, <em>p</em> &lt; 0.001), which suggested large amount of unexplained variations in true effects. Further, this unexplained heterogeneity has large variance (<span class="math inline">\(\tau^2\)</span> = 0.001) and it reflects the real variations that are not explained by species as suggested by high value of <span class="math inline">\(I^2\)</span> = 98.8%.</p>
<div class="layout-chunk" data-layout="l-body">
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig5"></span>
<img src="meta-analysis_files/figure-html5/fig5-1.svg" alt="The average effect of mycorrhizal association on litter decomposition rate for each species included in the meta-analysis. The values represent the average difference in response ratios compared to the reference species (*Acer saccharum*) and the associated bars denote the 95% confidence intervals" width="768" />
<p class="caption">
Figure 5: The average effect of mycorrhizal association on litter decomposition rate for each species included in the meta-analysis. The values represent the average difference in response ratios compared to the reference species (<em>Acer saccharum</em>) and the associated bars denote the 95% confidence intervals
</p>
</div>
</div>
<p>Further, the examination of the studentised residuals revealed that none of the studies had a value larger than <span class="math inline">\(\pm\)</span> 2.91 and hence there was no indication of outliers in the context of this model. Similarly, the inspection of Cook’s distances indicated that none of the studies could be considered overly influential.</p>
<div class="layout-chunk" data-layout="l-body">

</div>
<h3 id="publication-bias-1">Publication bias</h3>
<p>The visual inspection of the funnel plot did not indicate the publication bias as the outcomes appeared to be symmetrically distributed around the average effect estimate (Figure <a href="#fig:fig6">6</a>). The outcomes were well-fitted within the funnel as larger outcomes (with higher precision) were closely clustered at top whereas smaller outcomes (with lower precision) were dispersed around the average effect at bottom. Further, the funnel plot included nearly half outcomes in the non-significant region again suggesting absence of publication bias due to null findings of studies.</p>
<div class="layout-chunk" data-layout="l-body">
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig6"></span>
<img src="meta-analysis_files/figure-html5/fig6-1.svg" alt="The contour-enhanced funnel plot shows the effect size of each study (expressed as the response ratio) on the x-axis against a measure of their standard error (from large to small) on the y-axis. The vertical lines in the middle of the funnel show the average effect estimate and associated 95% confidence interval. The shaded region indicates the different significance levels for the estimated effect size for each outcome" width="768" />
<p class="caption">
Figure 6: The contour-enhanced funnel plot shows the effect size of each study (expressed as the response ratio) on the x-axis against a measure of their standard error (from large to small) on the y-axis. The vertical lines in the middle of the funnel show the average effect estimate and associated 95% confidence interval. The shaded region indicates the different significance levels for the estimated effect size for each outcome
</p>
</div>
</div>
<p>The intercept estimated from the regression test (<span class="math inline">\(\beta_0\)</span> = 0.06; <em>95% CI</em>: 0.016, 0.104) was not significantly higher than zero (<em>Z</em> = 0.51, <em>p</em> = 0.609). Also, there was non-significant low rank correlation (Kendall’s <span class="math inline">\(\tau\)</span> = -0.27, <em>p</em> = 0.193) between standard error and response ratio of outcomes. Thus, both the statistical tests for funnel asymmetry also corroborated our initial visual observations of funnel plot symmetry, which can be considered as indication of negligible or no evidence of publication bias for the present meta-analysis. (Figure <a href="#fig:fig6">6</a>).</p>
<p>The Rosenthal’s <em>fail-safe N</em> of 2107 suggested that over 2,000 outcomes with average response ratio of 1.0 need to be added to make the average effect non-significant (<em>p</em> = 0.05). Similarly, the Rosenberg’s <em>fail-safe N</em> was 4019, indicating that there would need to be over 4,000 outcomes with a mean response ratio of 1.0 added to the analysis, before the cumulative effect would become statistically non-significant (<em>p</em> = 0.05). Considering that we were able to identify only 14 relevant outcomes, it is unlikely that we missed over 4,000 or over 2,000 outcomes. Thus, although it may be possible that we might have estimated biased effect, it is unlikely that the actual overall effect is zero. However, we acknowledge that it is just one form of publication bias among the several other sources of bias in the estimate.</p>
<h2 id="discussion">Discussion</h2>
<p>Considering the initially posed questions, our analysis found moderate to strong evidence that mycorrhizal presence indeed affects the rate of litter decomposition in forest ecosystems (Question 1). The overall average estimate suggested that mycorrhizal fungi tends to enhance the litter decomposition rate by 7-8% on average (Question 2), though it can vary from a decrease of about 10% to an increase of over 30% in presence of mycorrhizal fungi. As with other ecological meta-analyses, large variations in effects were observed as indicated by potentially large heterogeneity. A small proportion (~30%) of these variations can be attributed to differences in species specific traits, possibly through litter quality (Question 3). However, no statistical evidence was found for the differences between angiosperm and gymnosperm species, though slightly higher enhancement was observed for angiosperms than the gymnosperms (Question 4). Further, these effects did not vary due to phylogenetic relationship among the included species (Question 3).</p>
<p>In contrast to expectations of Gadgil effect, our findings suggested an overall increase in the litter decomposition rates in presence of mycorrhizal fungi, which is consistent with previous findings <span class="citation" data-cites="Shah2016 Gui2017 Frey2019 Lang2021">(<a href="#ref-Shah2016" role="doc-biblioref">Shah et al. 2016</a>; <a href="#ref-Gui2017" role="doc-biblioref">Gui et al. 2017</a>; <a href="#ref-Frey2019" role="doc-biblioref">Frey 2019</a>; <a href="#ref-Lang2021" role="doc-biblioref">Lang et al. 2021</a>)</span>. The enhanced litter decomposition rates have been observed in the presence of both ectomycorrhizal <span class="citation" data-cites="Shah2016">(<a href="#ref-Shah2016" role="doc-biblioref">Shah et al. 2016</a>)</span> and arbuscular mycorrhizal fungi <span class="citation" data-cites="Gui2017">(<a href="#ref-Gui2017" role="doc-biblioref">Gui et al. 2017</a>)</span>. The ectomycorrhizal-mediated enhancement was suggested to be associated with the secretion of enzymes to acquire nutrients from organic matter <span class="citation" data-cites="Jackson2019">(<a href="#ref-Jackson2019" role="doc-biblioref">Jackson et al. 2019</a>)</span>. These enzymes degrade the complex compounds and make them available to other decomposer organisms. However, the increased litter decomposition in the presence of arbuscular mycorrhiza was attributed to their saprotrophic capabilities, which tend to acquire nutrients directly from organic matter <span class="citation" data-cites="Hodge2001 Herman2012">(<a href="#ref-Hodge2001" role="doc-biblioref">Hodge et al. 2001</a>; <a href="#ref-Herman2012" role="doc-biblioref">Herman et al. 2012</a>)</span>. Further, both ectomycorrhizal <span class="citation" data-cites="Brzostek2015">(<a href="#ref-Brzostek2015" role="doc-biblioref">Brzostek et al. 2015</a>)</span> and arbuscular mycorrhizal fungi <span class="citation" data-cites="Herman2012">(<a href="#ref-Herman2012" role="doc-biblioref">Herman et al. 2012</a>)</span> are observed to increase the activity of microbial communities by providing plant-based carbon sources, i.e., priming effects. Since microbial communities are associated with the breakdown of complex compounds and the release of nutrients, their increased activity enhances the litter decomposition rate <span class="citation" data-cites="Schneider2012">(<a href="#ref-Schneider2012" role="doc-biblioref">Schneider et al. 2012</a>)</span>. Thus, our findings question the generality of ‘Gadgil effect’ and suggest that the magnitude and direction of ‘Gadgil effect’ can be highly inconsistent and variable <span class="citation" data-cites="Fernandez2016">(<a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span>.</p>
<p>The much wider range of average effect (-10% to 30%) in our analysis suggest that multiple variables are involved in deriving the mycorrhizal effects on litter decomposition. Such a wider range is expected because the rate of litter decomposition is not controlled by mycorrhizal effects alone and several factors including climate, litter quality and soil decomposer organisms are involved in determining the decomposition rates <span class="citation" data-cites="Zhang2008 Cornwell2008 Kumar2021">(<a href="#ref-Zhang2008" role="doc-biblioref">Zhang et al. 2008</a>; <a href="#ref-Cornwell2008" role="doc-biblioref">Cornwell et al. 2008</a>; <a href="#ref-Kumar2021" role="doc-biblioref">Kumar et al. 2021</a>)</span>. Thus, the mycorrhizal-mediated litter decomposition rates may either hamper or stimulate depending upon the stage of decomposition, soil profile, climatic condition, plant species, and fungal species <span class="citation" data-cites="Fernandez2016 Sterkenburg2018">(<a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>; <a href="#ref-Sterkenburg2018" role="doc-biblioref">Sterkenburg et al. 2018</a>)</span>. The species-specific effects are observed in our analysis, as species identity appeared to explain a small part of total variations in the mycorrhizal effects. This species-specific variation in mycorrhizal effects can be attributed to the resource allocation strategy of plant species. When mycorrhizal fungi receive more carbon from the host plant, they can invest more in enzymes and secondary metabolites <span class="citation" data-cites="Rineau2013">(<a href="#ref-Rineau2013" role="doc-biblioref">Rineau et al. 2013</a>)</span>. This higher investment can directly increase the enzymatic degradation capacity of mycorrhizal fungi leading to faster decomposition rates. Further, high resource allocation can result in higher production of labile exudates leading to stimulation of growth and activity of saprotrophic microbes (priming effects).</p>
<p>Further, the species identity can indirectly affect mycorrhizal effects through the quality of litter produced by a species <span class="citation" data-cites="Cornwell2008 Patil2020srep">(<a href="#ref-Cornwell2008" role="doc-biblioref">Cornwell et al. 2008</a>; <a href="#ref-Patil2020srep" role="doc-biblioref">Patil et al. 2020a</a>)</span>. It is observed that ‘high quality’ litter (low lignin and high nitrogen) decomposes more rapidly than the ‘low quality’ litter (high lignin, low nitrogen) <span class="citation" data-cites="Patil2020srep">(<a href="#ref-Patil2020srep" role="doc-biblioref">Patil et al. 2020a</a>)</span>. Suppose a species will produce ‘high quality’ litter (low lignin and high nitrogen). In that case, mycorrhizal effects will be weaker due to more activities of saprotrophic fungi and decomposer micro-organisms whereas ‘low quality’ litter (high lignin, low nitrogen and high carbon content) will favour strong mycorrhizal effects as it will become accessible to the mycorrhizal fungi. Thus, it seems that various factors modulate the effects of mycorrhiza on litter decomposition rate; therefore, these effects can be highly context-dependent <span class="citation" data-cites="Fernandez2016">(<a href="#ref-Fernandez2016" role="doc-biblioref">Fernandez and Kennedy 2016</a>)</span>.</p>
<p>Although our meta-analysis supports the mycorrhizal-mediated faster decomposition rates, it suffers from some limitations and caveats. We expect that these limitations and caveats will inspire future investigations to advance our understanding about the role of mycorrhizal fungi in nutrient cycling. One of the major limitation of the present analysis is that the systematic literature search and screening resulted in a small number of comparable studies. Although the practice of systematic literature search and screening substantially improve the reproducibility of overall analysis, it may hamper us to identify all the evidence from the literature. Further, our strict eligibility criteria to identify comparable studies appeared to be overly narrower. Therefore, we suggest future research synthesis to include all possible studies across wider range of plant species and ecosystem types. Further, litter decomposition in natural ecosystems is regulated by a complex interaction of several factors (e.g., climate, soil, plant, and mycorrhizal fungi). However, our limited database did not allowed us to test the effects of these variables due to insufficient sample size. Furthermore, tropical forests are under-represented in our analysis, it may be possible that observed positive effects of mycorrhizal fungi are localised to temperate forests <span class="citation" data-cites="Keller2019">(<a href="#ref-Keller2019" role="doc-biblioref">Keller and Phillips 2019</a>)</span>. Similarly, all the eligible studies considered only leaf litter for decomposition, the inclusion of litter from other sources might lead to variation in the results. Moreover, we have not considered the actual sample size, mean, and variation of measurements; instead, we considered the number of data points as the sample size to ensure the consistency among the studies. Finally, as with any systematic exercise, deviation from the systematic protocol (searching keywords, eligibility criteria) may lead to different results.</p>
<p>We encourage researchers to consider the known limitations of the present study. We observed that most studies considered leaf litter and litter from other sources (root, shoot, and wood) is lacking. Thus, there is a need to experiment with other litter along with leaf litter. Further, we suggest the future investigations to study the mycorrhizal effects on litter decomposition rates across biomes (grassland, heathland, temperate forest and tropical forest), plant species (angiosperm vs gymnosperm, exotic vs native species, leguminous vs non leguminous species, evergreen vs deciduous), litter type (fine roots vs leaf litter, high quality litter vs low quality litter), and mycorrhizal groups (arbuscular mycorrhizal fungi, ectomycorrhizal fungi, ericoid mycorrhizal fungi, orchid mycorrhizal fungi). Moreover, we need to investigate these effects along the gradients of climate, elevation and abundance of mycorrhizal fungi (i.e., trenching depth).</p>
<h2 id="conclusion">Conclusion</h2>
<p>Overall, our meta-analysis suggested that the presence of mycorrhizal fungi tends to increase the litter decomposition rates in forest ecosystems, which questioned the generality of ‘Gadgil effect.’ Further, high variability of magnitude and direction across species indicates the involvement of multiple mechanisms. Therefore, we also caution about the generalizability and interpret-ability of such effects and processes. We need to improve our understanding of what, when, and how mycorrhiza regulates litter decomposition. This kind of information will be helpful to understand and model the responses of nutrient cycling under the influence of changing environment. However, there is an apparent lack of evidence, and we need more data further to understand the role of mycorrhiza in the litter decomposition mechanism. We believe the present analysis will stimulate further research to advance our knowledge about the roles of mycorrhiza in the regulation of nutrient cycling across various scales.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r distill-force-highlighting-css"><code class="sourceCode r"></code></pre></div>
<div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-Averill2014" class="csl-entry" role="doc-biblioentry">
Averill C, Turner BL, Finzi AC (2014) <span class="nocase">Mycorrhiza-mediated competition between plants and decomposers drives soil carbon storage</span>. Nature 505:543–545. <a href="https://doi.org/10.1038/nature12901">https://doi.org/10.1038/nature12901</a>
</div>
<div id="ref-Beeck2018" class="csl-entry" role="doc-biblioentry">
Beeck MOD, Troein C, Peterson C, et al (2018) Fenton reaction facilitates organic nitrogen acquisition by an ectomycorrhizal fungus. New Phytologist 218:335–343. <a href="https://doi.org/10.1111/nph.14971">https://doi.org/10.1111/nph.14971</a>
</div>
<div id="ref-Begg1994" class="csl-entry" role="doc-biblioentry">
Begg CB, Mazumdar M (1994) Operating characteristics of a rank correlation test for publication bias. Biometrics 50:1088. <a href="https://doi.org/10.2307/2533446">https://doi.org/10.2307/2533446</a>
</div>
<div id="ref-Borenstein2009" class="csl-entry" role="doc-biblioentry">
Borenstein M, Hedges LV, Higgins JPT, Rothstein HR (2009) Introduction to meta-analysis, 1st edn. John Wiley <span>&amp;</span> Sons, Ltd
</div>
<div id="ref-Borenstein2010" class="csl-entry" role="doc-biblioentry">
Borenstein M, Hedges LV, Higgins JPT, Rothstein HR (2010) <span class="nocase">A basic introduction to fixed-effect and random-effects models for meta-analysis</span>. Research Synthesis Methods 1:97–111. <a href="https://doi.org/10.1002/jrsm.12">https://doi.org/10.1002/jrsm.12</a>
</div>
<div id="ref-Brundrett2018" class="csl-entry" role="doc-biblioentry">
Brundrett MC, Tedersoo L (2018) <span class="nocase">Evolutionary history of mycorrhizal symbioses and global host plant diversity</span>. New Phytologist 220:1108–1115. <a href="https://doi.org/10.1111/nph.14976">https://doi.org/10.1111/nph.14976</a>
</div>
<div id="ref-Brzostek2015" class="csl-entry" role="doc-biblioentry">
Brzostek ER, Dragoni D, Brown ZA, Phillips RP (2015) <span class="nocase">Mycorrhizal type determines the magnitude and direction of root‐induced changes in decomposition in a temperate forest</span>. New Phytologist 206:1274–1282. <a href="https://doi.org/10.1111/nph.13303">https://doi.org/10.1111/nph.13303</a>
</div>
<div id="ref-Cinar2021" class="csl-entry" role="doc-biblioentry">
Cinar O, Nakagawa S, Viechtbauer W (2021) Phylogenetic multilevel meta-analysis: A simulation study on the importance of modelling the phylogeny. Methods in Ecology and Evolution 13:383–395. <a href="https://doi.org/10.1111/2041-210x.13760">https://doi.org/10.1111/2041-210x.13760</a>
</div>
<div id="ref-Cochran1954" class="csl-entry" role="doc-biblioentry">
Cochran WG (1954) <span class="nocase">The combination of estimates from different experiments</span>. Biometrics 10:101. <a href="https://doi.org/10.2307/3001666">https://doi.org/10.2307/3001666</a>
</div>
<div id="ref-Cooper2009" class="csl-entry" role="doc-biblioentry">
Cooper H, Hedges LV, Valentine JC (eds) (2009) <span class="nocase">The handbook of research synthesis and meta-analysis</span>, 2nd edn. Russell Sage Foundation, New York
</div>
<div id="ref-Cornwell2008" class="csl-entry" role="doc-biblioentry">
Cornwell WK, Cornelissen JHCC, Amatangelo K, et al (2008) <span class="nocase">Plant species traits are the predominant control on litter decomposition rates within biomes worldwide</span>. Ecology Letters 11:1065–1071. <a href="https://doi.org/10.1111/j.1461-0248.2008.01219.x">https://doi.org/10.1111/j.1461-0248.2008.01219.x</a>
</div>
<div id="ref-Entry1991" class="csl-entry" role="doc-biblioentry">
Entry JA, Rose CL, Cromack K (1991) <span class="nocase">Litter decomposition and nutrient release in ectomycorrhizal mat soils of a Douglas fir ecosystem</span>. Soil Biology and Biochemistry 23:285–290. <a href="https://doi.org/10.1016/0038-0717(91)90065-R">https://doi.org/10.1016/0038-0717(91)90065-R</a>
</div>
<div id="ref-Fernandez2016" class="csl-entry" role="doc-biblioentry">
Fernandez CW, Kennedy PG (2016) <span class="nocase">Revisiting the ‘Gadgil effect’: do interguild fungal interactions control carbon cycling in forest soils?</span> New Phytologist 209:1382–1394. <a href="https://doi.org/10.1111/nph.13648">https://doi.org/10.1111/nph.13648</a>
</div>
<div id="ref-Fernandez2020" class="csl-entry" role="doc-biblioentry">
Fernandez CW, See CR, Kennedy PG (2020) <span class="nocase">Decelerated carbon cycling by ectomycorrhizal fungi is controlled by substrate quality and community composition</span>. New Phytologist 226:569–582. <a href="https://doi.org/10.1111/nph.16269">https://doi.org/10.1111/nph.16269</a>
</div>
<div id="ref-Finlay2005" class="csl-entry" role="doc-biblioentry">
Finlay R (2005) <span class="nocase">Action and interaction in the mycorrhizal hyphosphere — A re-evaluation of the role of mycorrhizas in nutrient acquisition and plant ecology</span>. In: Nutrient acquisition by plants. Springer-Verlag, Berlin/Heidelberg, pp 221–276
</div>
<div id="ref-Frey2019" class="csl-entry" role="doc-biblioentry">
Frey SD (2019) Mycorrhizal fungi as mediators of soil organic matter dynamics. Annual Review of Ecology, Evolution, and Systematics 50:237–259. <a href="https://doi.org/10.1146/annurev-ecolsys-110617-062331">https://doi.org/10.1146/annurev-ecolsys-110617-062331</a>
</div>
<div id="ref-Gadgil1975" class="csl-entry" role="doc-biblioentry">
Gadgil RL, Gadgil PD (1975) Suppression of litter decomposition by mycorrhizal roots of <span class="nocase">Pinus radiata</span>. New Zealand Journal of Forestry Science 5:33–41
</div>
<div id="ref-Gadgil1971" class="csl-entry" role="doc-biblioentry">
Gadgil RL, Gadgil PD (1971) <span class="nocase">Mycorrhiza and litter decomposition</span>. Nature 233:133. <a href="https://doi.org/10.1038/233133a0">https://doi.org/10.1038/233133a0</a>
</div>
<div id="ref-Gorka2019" class="csl-entry" role="doc-biblioentry">
Gorka S, Dietrich M, Mayerhofer W, et al (2019) <span class="nocase">Rapid transfer of plant photosynthates to soil bacteria via ectomycorrhizal hyphae and its interaction with nitrogen availability</span>. Frontiers in Microbiology 10:168. <a href="https://doi.org/10.3389/fmicb.2019.00168">https://doi.org/10.3389/fmicb.2019.00168</a>
</div>
<div id="ref-Grafen1989" class="csl-entry" role="doc-biblioentry">
Grafen A (1989) The phylogenetic regression. Philosophical Transactions of the Royal Society of London B, Biological Sciences 326:119–157. <a href="https://doi.org/10.1098/rstb.1989.0106">https://doi.org/10.1098/rstb.1989.0106</a>
</div>
<div id="ref-Gui2017" class="csl-entry" role="doc-biblioentry">
Gui H, Hyde K, Xu J, Mortimer P (2017) <span class="nocase">Arbuscular mycorrhiza enhance the rate of litter decomposition while inhibiting soil microbial community development</span>. Scientific Reports 7:42184. <a href="https://doi.org/10.1038/srep42184">https://doi.org/10.1038/srep42184</a>
</div>
<div id="ref-Haddaway2018" class="csl-entry" role="doc-biblioentry">
Haddaway NR, Macura B, Whaley P, Pullin AS (2018) <span class="nocase">ROSES Reporting standards for Systematic Evidence Syntheses: Pro forma, flow-diagram and descriptive summary of the plan and conduct of environmental systematic reviews and systematic maps</span>. Environmental Evidence 7:1–8. <a href="https://doi.org/10.1186/s13750-018-0121-7">https://doi.org/10.1186/s13750-018-0121-7</a>
</div>
<div id="ref-Harrer2021" class="csl-entry" role="doc-biblioentry">
Harrer M, Cuijpers P, Furukawa TA, Ebert DD (2021) <span class="nocase">Doing Meta-Analysis with R</span>. Chapman; Hall/CRC, Boca Raton
</div>
<div id="ref-Hedges1999" class="csl-entry" role="doc-biblioentry">
Hedges LV, Gurevitch J, Curtis PS (1999) The meta-analysis of response ratios in experimental ecology. Ecology 80:1150–1156. <a href="https://doi.org/10.1890/0012-9658(1999)080[1150:tmaorr]2.0.co;2">https://doi.org/10.1890/0012-9658(1999)080[1150:tmaorr]2.0.co;2</a>
</div>
<div id="ref-Heijden2015" class="csl-entry" role="doc-biblioentry">
Heijden MGA, Martin FM, Selosse M, Sanders IR (2015) <span class="nocase">Mycorrhizal ecology and evolution: the past, the present, and the future</span>. New Phytologist 205:1406–1423. <a href="https://doi.org/10.1111/nph.13288">https://doi.org/10.1111/nph.13288</a>
</div>
<div id="ref-Herman2012" class="csl-entry" role="doc-biblioentry">
Herman DJ, Firestone MK, Nuccio E, Hodge A (2012) <span class="nocase">Interactions between an arbuscular mycorrhizal fungus and a soil microbial community mediating litter decomposition</span>. FEMS Microbiology Ecology 80:236–247. <a href="https://doi.org/10.1111/j.1574-6941.2011.01292.x">https://doi.org/10.1111/j.1574-6941.2011.01292.x</a>
</div>
<div id="ref-Higgins2002" class="csl-entry" role="doc-biblioentry">
Higgins JPT, Thompson SG (2002) <span class="nocase">Quantifying heterogeneity in a meta-analysis</span>. Statistics in Medicine 21:1539–1558. <a href="https://doi.org/10.1002/sim.1186">https://doi.org/10.1002/sim.1186</a>
</div>
<div id="ref-Hodge2001" class="csl-entry" role="doc-biblioentry">
Hodge A, Campbell CD, Fitter AH (2001) <span class="nocase">An arbuscular mycorrhizal fungus accelerates decomposition and acquires nitrogen directly from organic material</span>. Nature 413:297–299. <a href="https://doi.org/10.1038/35095041">https://doi.org/10.1038/35095041</a>
</div>
<div id="ref-Jackson2019" class="csl-entry" role="doc-biblioentry">
Jackson O, Quilliam RS, Stott A, et al (2019) <span class="nocase">Rhizosphere carbon supply accelerates soil organic matter decomposition in the presence of fresh organic substrates</span>. Plant and Soil 440:473–490. <a href="https://doi.org/10.1007/s11104-019-04072-3">https://doi.org/10.1007/s11104-019-04072-3</a>
</div>
<div id="ref-Jin2019" class="csl-entry" role="doc-biblioentry">
Jin Y, Qian H (2019) <span>V.PhyloMaker</span>: An <span>R</span> package that can generate very large phylogenies for vascular plants. Ecography 42:1353–1359. <a href="https://doi.org/10.1111/ecog.04434">https://doi.org/10.1111/ecog.04434</a>
</div>
<div id="ref-Keller2019" class="csl-entry" role="doc-biblioentry">
Keller AB, Phillips RP (2019) Leaf litter decay rates differ between mycorrhizal groups in temperate, but not tropical, forests. New Phytologist 222:556–564. <a href="https://doi.org/10.1111/nph.15524">https://doi.org/10.1111/nph.15524</a>
</div>
<div id="ref-Koide2003" class="csl-entry" role="doc-biblioentry">
Koide RT, Wu T (2003) <span class="nocase">Ectomycorrhizas and retarded decomposition in a Pinus resinosa plantation</span>. New Phytologist 158:401–407. <a href="https://doi.org/10.1046/j.1469-8137.2003.00732.x">https://doi.org/10.1046/j.1469-8137.2003.00732.x</a>
</div>
<div id="ref-Koricheva2014" class="csl-entry" role="doc-biblioentry">
Koricheva J, Gurevitch J (2014) Uses and misuses of meta-analysis in plant ecology. Journal of Ecology 102:828–844. <a href="https://doi.org/10.1111/1365-2745.12224">https://doi.org/10.1111/1365-2745.12224</a>
</div>
<div id="ref-Krywolap1964" class="csl-entry" role="doc-biblioentry">
Krywolap GN, Grand LF, Casida Jr. LE (1964) <span class="nocase">The natural occurrence of an antibiotic in the mycorrhizal fungus</span>. Canadian Journal of Microbiology 10:323–328. <a href="https://doi.org/10.1139/m64-044">https://doi.org/10.1139/m64-044</a>
</div>
<div id="ref-Kubicek2011" class="csl-entry" role="doc-biblioentry">
Kubicek CP, Herrera-Estrella A, Seidl-Seiboth V, et al (2011) <span class="nocase">Comparative genome sequence analysis underscores mycoparasitism as the ancestral life style of Trichoderma</span>. Genome Biology 12:R40. <a href="https://doi.org/10.1186/gb-2011-12-4-r40">https://doi.org/10.1186/gb-2011-12-4-r40</a>
</div>
<div id="ref-Kumar2022" class="csl-entry" role="doc-biblioentry">
Kumar A, Patil M, Kumar P, et al (2022) Plant ecology in indian siwalik range: A systematic map and its bibliometric analysis. Tropical Ecology. <a href="https://doi.org/10.1007/s42965-022-00229-x">https://doi.org/10.1007/s42965-022-00229-x</a>
</div>
<div id="ref-Kumar2021" class="csl-entry" role="doc-biblioentry">
Kumar A, Patil M, Kumar P, Singh AN (2021) <span class="nocase">Phosphorus and litter decomposability traits in tropical forest ecosystems under changing environment: A synthesis</span>. In: Chaturvedi RK, Singh R, Bhadouria R (eds) Tropical dry forests: Emerging features and ecological perspectives. Nova Science Publishers, Inc., New York, NY, pp 311–336
</div>
<div id="ref-Lajeunesse2015" class="csl-entry" role="doc-biblioentry">
Lajeunesse MJ (2015) Bias and correction for the log response ratio in ecological meta-analysis. Ecology 96:2056–2063. <a href="https://doi.org/10.1890/14-2402.1">https://doi.org/10.1890/14-2402.1</a>
</div>
<div id="ref-Lang2021" class="csl-entry" role="doc-biblioentry">
Lang AK, Jevon FV, Vietorisz CR, et al (2021) <span class="nocase">Fine roots and mycorrhizal fungi accelerate leaf litter decomposition in a northern hardwood forest regardless of dominant tree mycorrhizal associations</span>. New Phytologist 230:316–326. <a href="https://doi.org/10.1111/nph.17155">https://doi.org/10.1111/nph.17155</a>
</div>
<div id="ref-Leifheit2015" class="csl-entry" role="doc-biblioentry">
Leifheit EF, Verbruggen E, Rillig MC (2015) <span class="nocase">Arbuscular mycorrhizal fungi reduce decomposition of woody plant litter while increasing soil aggregation</span>. Soil Biology and Biochemistry 81:323–328. <a href="https://doi.org/10.1016/j.soilbio.2014.12.003">https://doi.org/10.1016/j.soilbio.2014.12.003</a>
</div>
<div id="ref-Mayor2006" class="csl-entry" role="doc-biblioentry">
Mayor JR, Henkel TW (2006) <span class="nocase">Do ectomycorrhizas alter leaf-litter decomposition in monodominant tropical forests of Guyana?</span> New Phytologist 169:579–588. <a href="https://doi.org/10.1111/j.1469-8137.2005.01607.x">https://doi.org/10.1111/j.1469-8137.2005.01607.x</a>
</div>
<div id="ref-McGuire2010" class="csl-entry" role="doc-biblioentry">
McGuire KL, Treseder KK (2010) <span class="nocase">Microbial communities and their relevance for ecosystem models: Decomposition as a case study</span>. Soil Biology and Biochemistry 42:529–535. <a href="https://doi.org/10.1016/j.soilbio.2009.11.016">https://doi.org/10.1016/j.soilbio.2009.11.016</a>
</div>
<div id="ref-Nakagawa2017" class="csl-entry" role="doc-biblioentry">
Nakagawa S, Noble DWA, Senior AM, Lagisz M (2017) Meta-evaluation of meta-analysis: Ten appraisal questions for biologists. <span>BMC</span> Biology 15: <a href="https://doi.org/10.1186/s12915-017-0357-7">https://doi.org/10.1186/s12915-017-0357-7</a>
</div>
<div id="ref-Nakagawa2012" class="csl-entry" role="doc-biblioentry">
Nakagawa S, Santos ESA (2012) Methodological issues and advances in biological meta-analysis. Evolutionary Ecology 26:1253–1274. <a href="https://doi.org/10.1007/s10682-012-9555-5">https://doi.org/10.1007/s10682-012-9555-5</a>
</div>
<div id="ref-Olson1963" class="csl-entry" role="doc-biblioentry">
Olson JS (1963) <span class="nocase">Energy storage and the balance of producers and decomposers in ecological systems</span>. Ecology 44:322–331. <a href="https://doi.org/10.2307/1932179">https://doi.org/10.2307/1932179</a>
</div>
<div id="ref-Orwin2011" class="csl-entry" role="doc-biblioentry">
Orwin KH, Kirschbaum MUF, St John MG, Dickie IA (2011) <span class="nocase">Organic nutrient uptake by mycorrhizal fungi enhances ecosystem carbon storage: a model-based assessment</span>. Ecology Letters 14:493–502. <a href="https://doi.org/10.1111/j.1461-0248.2011.01611.x">https://doi.org/10.1111/j.1461-0248.2011.01611.x</a>
</div>
<div id="ref-Pan2011" class="csl-entry" role="doc-biblioentry">
Pan Y, Birdsey RA, Fang J, et al (2011) <span class="nocase">A large and persistent carbon sink in the world’s forests</span>. Science 333:988–993. <a href="https://doi.org/10.1126/science.1201609">https://doi.org/10.1126/science.1201609</a>
</div>
<div id="ref-ape2019" class="csl-entry" role="doc-biblioentry">
Paradis E, Schliep K (2019) <span class="nocase">ape 5.0</span>: An environment for modern phylogenetics and evolutionary analyses in <span>R</span>. Bioinformatics 35:526–528
</div>
<div id="ref-Patil2020srep" class="csl-entry" role="doc-biblioentry">
Patil M, Kumar A, Kumar P, et al (2020a) Comparative litter decomposability traits of selected native and exotic woody species from an urban environment of north-western siwalik region, india. Scientific Reports 10:7888. <a href="https://doi.org/10.1038/s41598-020-64576-2">https://doi.org/10.1038/s41598-020-64576-2</a>
</div>
<div id="ref-Patil2020azb" class="csl-entry" role="doc-biblioentry">
Patil M, Kumar A, Kumar P, Singh AN (2020b) <span class="nocase">Litter decomposability traits and their linkage with the cycling of nutrients in the forest ecosystems under the lens of climate change: A conceptual overview</span>. Advances in Zoology and Botany 8:269–277. <a href="https://doi.org/10.13189/azb.2020.080324">https://doi.org/10.13189/azb.2020.080324</a>
</div>
<div id="ref-Pullin2018" class="csl-entry" role="doc-biblioentry">
Pullin AS, Frampton GK, Livoreil B, Petrokofsky G (eds) (2018) <span class="nocase">Guidelines and Standards for Evidence Synthesis in Environmental Management.</span>, 5th edn. https://environmentalevidence.org/information-for-authors/
</div>
<div id="ref-Qian2016" class="csl-entry" role="doc-biblioentry">
Qian H, Jin Y (2016) An updated megaphylogeny of plants, a tool for generating plant phylogenies and an analysis of phylogenetic community structure. Journal of Plant Ecology 9:233–239. <a href="https://doi.org/10.1093/jpe/rtv047">https://doi.org/10.1093/jpe/rtv047</a>
</div>
<div id="ref-R2021" class="csl-entry" role="doc-biblioentry">
R Core Team (2021) <span class="nocase">R: A language and environment for statistical computing</span>. R Foundation for Statistical Computing, Vienna, Austria
</div>
<div id="ref-Rasanayagam1992" class="csl-entry" role="doc-biblioentry">
Rasanayagam S, Jeffries P (1992) <span class="nocase">Production of acid is responsible for antibiosis by some ectomycorrhizal fungi</span>. Mycological Research 96:971–976. <a href="https://doi.org/10.1016/S0953-7562(09)80600-X">https://doi.org/10.1016/S0953-7562(09)80600-X</a>
</div>
<div id="ref-Rineau2013" class="csl-entry" role="doc-biblioentry">
Rineau F, Shah F, Smits MM, et al (2013) <span class="nocase">Carbon availability triggers the decomposition of plant litter and assimilation of nitrogen by an ectomycorrhizal fungus</span>. ISME Journal 7:2010–2022. <a href="https://doi.org/10.1038/ismej.2013.91">https://doi.org/10.1038/ismej.2013.91</a>
</div>
<div id="ref-Rosenberg2005" class="csl-entry" role="doc-biblioentry">
Rosenberg MS (2005) The file-drawer problem revisited: A general weighted method for calculating fail-safe numbers in meta-analysis. Evolution 59:464–468. <a href="https://doi.org/10.1111/j.0014-3820.2005.tb01004.x">https://doi.org/10.1111/j.0014-3820.2005.tb01004.x</a>
</div>
<div id="ref-Rosenthal1979" class="csl-entry" role="doc-biblioentry">
Rosenthal R (1979) The file drawer problem and tolerance for null results. Psychological Bulletin 86:638–641. <a href="https://doi.org/10.1037/0033-2909.86.3.638">https://doi.org/10.1037/0033-2909.86.3.638</a>
</div>
<div id="ref-Schneider2012" class="csl-entry" role="doc-biblioentry">
Schneider T, Keiblinger KM, Schmid E, et al (2012) <span class="nocase">Who is who in litter decomposition Metaproteomics reveals major microbial players and their biogeochemical functions</span>. ISME Journal 6:1749–1762. <a href="https://doi.org/10.1038/ismej.2012.11">https://doi.org/10.1038/ismej.2012.11</a>
</div>
<div id="ref-Senior2016" class="csl-entry" role="doc-biblioentry">
Senior AM, Grueber CE, Kamiya T, et al (2016) Heterogeneity in ecological and evolutionary meta-analyses: Its magnitude and implications. Ecology 97:3293–3299. <a href="https://doi.org/10.1002/ecy.1591">https://doi.org/10.1002/ecy.1591</a>
</div>
<div id="ref-Shah2016" class="csl-entry" role="doc-biblioentry">
Shah F, Nicolás C, Bentzer J, et al (2016) <span class="nocase">Ectomycorrhizal fungi decompose soil organic matter using oxidative mechanisms adapted from saprotrophic ancestors</span>. New Phytologist 209:1705–1719. <a href="https://doi.org/10.1111/NPH.13722">https://doi.org/10.1111/NPH.13722</a>
</div>
<div id="ref-Shu2019" class="csl-entry" role="doc-biblioentry">
Shu M, Zhao Q, Li Z, et al (2019) <span class="nocase">Effects of global change factors and living roots on root litter decomposition in a Qinghai-Tibet alpine meadow</span>. Scientific Reports 9:16924. <a href="https://doi.org/10.1038/s41598-019-53450-5">https://doi.org/10.1038/s41598-019-53450-5</a>
</div>
<div id="ref-Smith2018" class="csl-entry" role="doc-biblioentry">
Smith SA, Brown JW (2018) Constructing a broadly inclusive seed plant phylogeny. American Journal of Botany 105:302–314. <a href="https://doi.org/10.1002/ajb2.1019">https://doi.org/10.1002/ajb2.1019</a>
</div>
<div id="ref-Staaf1988" class="csl-entry" role="doc-biblioentry">
Staaf H (1988) <span class="nocase">Litter decomposition in beech forests ? effects of excluding tree roots</span>. Biology and Fertility of Soils 6:302–305. <a href="https://doi.org/10.1007/BF00261017">https://doi.org/10.1007/BF00261017</a>
</div>
<div id="ref-Steidinger2019" class="csl-entry" role="doc-biblioentry">
Steidinger BS, Crowther TW, Liang J, et al (2019) <span class="nocase">Climatic controls of decomposition drive the global biogeography of forest-tree symbioses</span>. Nature 569:404–408. <a href="https://doi.org/10.1038/s41586-019-1128-0">https://doi.org/10.1038/s41586-019-1128-0</a>
</div>
<div id="ref-Sterkenburg2018" class="csl-entry" role="doc-biblioentry">
Sterkenburg E, Clemmensen KE, Ekblad A, et al (2018) <span class="nocase">Contrasting effects of ectomycorrhizal fungi on early and late stage decomposition in a boreal forest</span>. ISME Journal 12:2187–2197. <a href="https://doi.org/10.1038/s41396-018-0181-2">https://doi.org/10.1038/s41396-018-0181-2</a>
</div>
<div id="ref-Sterne2006" class="csl-entry" role="doc-biblioentry">
Sterne JAC, Egger M (2006) Regression methods to detect publication and other bias in meta-analysis. In: Rothstein HR, Sutton AJ, Borenstein M (eds) Publication bias in meta-analysis. John Wiley <span>&amp;</span> Sons, Ltd, pp 99–110
</div>
<div id="ref-Sulman2017" class="csl-entry" role="doc-biblioentry">
Sulman BN, Brzostek ER, Medici C, et al (2017) <span class="nocase">Feedbacks between plant N demand and rhizosphere priming depend on type of mycorrhizal association</span>. Ecology Letters 20:1043–1053. <a href="https://doi.org/10.1111/ele.12802">https://doi.org/10.1111/ele.12802</a>
</div>
<div id="ref-Talbot2015" class="csl-entry" role="doc-biblioentry">
Talbot JM, Martin F, Kohler A, et al (2015) <span class="nocase">Functional guild classification predicts the enzymatic role of fungi in litter and soil biogeochemistry</span>. Soil Biology and Biochemistry 88:441–456. <a href="https://doi.org/10.1016/j.soilbio.2015.05.006">https://doi.org/10.1016/j.soilbio.2015.05.006</a>
</div>
<div id="ref-Tedersoo2019" class="csl-entry" role="doc-biblioentry">
Tedersoo L, Bahram M (2019) Mycorrhizal types differ in ecophysiology and alter plant nutrition and soil processes. Biological Reviews 94:1857–1880. <a href="https://doi.org/10.1111/brv.12538">https://doi.org/10.1111/brv.12538</a>
</div>
<div id="ref-metafor2010" class="csl-entry" role="doc-biblioentry">
Viechtbauer W (2010) <span class="nocase">Conducting Meta-Analyses in R with the metafor Package</span>. Journal of Statistical Software 36:1–48. <a href="https://doi.org/10.18637/jss.v036.i03">https://doi.org/10.18637/jss.v036.i03</a>
</div>
<div id="ref-Viechtbauer2005" class="csl-entry" role="doc-biblioentry">
Viechtbauer W (2005) Bias and efficiency of meta-analytic variance estimators in the random-effects model. Journal of Educational and Behavioral Statistics 30:261–293. <a href="https://doi.org/10.3102/10769986030003261">https://doi.org/10.3102/10769986030003261</a>
</div>
<div id="ref-Viechtbauer2010" class="csl-entry" role="doc-biblioentry">
Viechtbauer W, Cheung MW-L (2010) Outlier and influence diagnostics for meta-analysis. Research Synthesis Methods 1:112–125. <a href="https://doi.org/10.1002/jrsm.11">https://doi.org/10.1002/jrsm.11</a>
</div>
<div id="ref-Westgate2020" class="csl-entry" role="doc-biblioentry">
Westgate M, Grames E (2020) <span class="nocase">synthesisr</span>: Import, assemble, and deduplicate bibliographic datasets
</div>
<div id="ref-Wickham2019" class="csl-entry" role="doc-biblioentry">
Wickham H, Averick M, Bryan J, et al (2019) <span class="nocase">Welcome to the Tidyverse</span>. Journal of Open Source Software 4:1686. <a href="https://doi.org/10.21105/joss.01686">https://doi.org/10.21105/joss.01686</a>
</div>
<div id="ref-ggtree2016" class="csl-entry" role="doc-biblioentry">
Yu G, Smith DK, Zhu H, et al (2016) <span class="nocase">ggtree</span>: An <span>R</span> package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Methods in Ecology and Evolution 8:28–36. <a href="https://doi.org/10.1111/2041-210x.12628">https://doi.org/10.1111/2041-210x.12628</a>
</div>
<div id="ref-Zanne2014" class="csl-entry" role="doc-biblioentry">
Zanne AE, Tank DC, Cornwell WK, et al (2014) Three keys to the radiation of angiosperms into freezing environments. Nature 506:89–92. <a href="https://doi.org/10.1038/nature12872">https://doi.org/10.1038/nature12872</a>
</div>
<div id="ref-Zhang2008" class="csl-entry" role="doc-biblioentry">
Zhang D, Hui D, Luo Y, Zhou G (2008) <span class="nocase">Rates of litter decomposition in terrestrial ecosystems: global patterns and controlling factors</span>. Journal of Plant Ecology 1:85–93. <a href="https://doi.org/10.1093/jpe/rtn002">https://doi.org/10.1093/jpe/rtn002</a>
</div>
<div id="ref-Zhu1996" class="csl-entry" role="doc-biblioentry">
Zhu W, Ehrenfeld JG (1996) <span class="nocase">The effects of mycorrhizal roots on litter decomposition, soil biota, and nutrients in a spodosolic soil</span>. Plant and Soil 179:109–118. <a href="https://doi.org/10.1007/BF00011648">https://doi.org/10.1007/BF00011648</a>
</div>
</div>
<!--radix_placeholder_article_footer-->
<!--/radix_placeholder_article_footer-->
</div>

<div class="d-appendix">
</div>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

<!--radix_placeholder_site_after_body-->
<!--/radix_placeholder_site_after_body-->
<!--radix_placeholder_appendices-->
<div class="appendix-bottom">
<h3 id="references">References</h3>
<div id="references-listing"></div>
</div>
<!--/radix_placeholder_appendices-->
<!--radix_placeholder_navigation_after_body-->
<!--/radix_placeholder_navigation_after_body-->

</body>

</html>
